# SOME DESCRIPTIVE TITLE.
# Copyright (C) YEAR Red Hat, Inc.
# This file is distributed under the same license as the PACKAGE package.
#
# Translators:
# Bartosz Sapijaszko <bartosz@sapek.net>, 2002.
# dcantrel <dcantrell@redhat.com>, 2011.
# Dimitris Glezos <glezos@indifex.com>, 2011.
# Leszek Matok <lam@lac.pl>, 2004.
# Pawel Szopinski <pawel@szopinski.co.uk>, 2004.
# Piotr Drąg <piotrdrag@gmail.com>, 2008,2011-2013.
# sonyam <sonyam@tlen.pl>, 2004.
# Stanisław Małolepszy <smalolepszy@aviary.pl>, 2006.
# Tom Berner <tom@lodz.pl>, 2005.
# Tom Berner <tom@man.lodz.pl>, 2004.
# Wojciech Kapusta <wojciech@aviary.pl>, 2006.
msgid ""
msgstr ""
"Project-Id-Version: Anaconda\n"
"Report-Msgid-Bugs-To: anaconda-devel-list@redhat.com\n"
"POT-Creation-Date: 2013-01-07 16:55-0800\n"
"PO-Revision-Date: 2013-01-05 00:04+0000\n"
"Last-Translator: Piotr Drąg <piotrdrag@gmail.com>\n"
"Language-Team: Polish <trans-pl@lists.fedoraproject.org>\n"
"Language: pl\n"
"MIME-Version: 1.0\n"
"Content-Type: text/plain; charset=UTF-8\n"
"Content-Transfer-Encoding: 8bit\n"
"Plural-Forms: nplurals=3; plural=(n==1 ? 0 : n%10>=2 && n%10<=4 && (n%100<10 "
"|| n%100>=20) ? 1 : 2);\n"

#: anaconda:344
#, c-format
msgid ""
"%s requires %s MB of memory to install, but you only have %s MB on this "
"machine.\n"
msgstr ""
"System %s wymaga %s MB pamięci do instalacji, ale komputer posiada tylko %s "
"MB.\n"

#: anaconda:346
#, c-format
msgid ""
"The %s graphical installer requires %s MB of memory, but you only have %s MB."
msgstr ""
"Graficzny instalator systemu %s wymaga %s MB pamięci, ale komputer posiada "
"tylko %s MB."

#: anaconda:351
msgid "Not enough RAM"
msgstr "Za mało pamięci RAM"

#: anaconda:352
msgid ""
" Try the text mode installer by running:\n"
"\n"
"'/usr/bin/liveinst -T'\n"
"\n"
" from a root terminal."
msgstr ""
" Można wypróbować instalator w trybie tekstowym wykonując:\n"
"\n"
"\"/usr/bin/liveinst -T\"\n"
"\n"
" z terminala roota."

#: anaconda:355
msgid " Starting text mode."
msgstr " Uruchamianie trybu tekstowego."

#: anaconda:382 pyanaconda/rescue.py:268 pyanaconda/rescue.py:301
#: pyanaconda/rescue.py:314 pyanaconda/rescue.py:382 pyanaconda/rescue.py:398
#: pyanaconda/rescue.py:409
msgid "OK"
msgstr "OK"

#: anaconda:484
msgid ""
"Text mode provides a limited set of installation options. It does not allow "
"you to specify your own partitioning layout or package selections. Would you "
"like to use VNC mode instead?"
msgstr ""
"Tryb tekstowy dostarcza ograniczony zestaw opcji instalacji. Nie pozwala na "
"podanie własnego układu partycjonowania ani wyboru pakietów. Użyć trybu VNC "
"zamiast tego?"

#: anaconda:508
msgid "DISPLAY variable not set. Starting text mode."
msgstr "Zmienna DISPLAY nie jest ustawiona. Uruchamianie trybu tekstowego."

#: anaconda:516
msgid "Graphical installation is not available. Starting text mode."
msgstr "Instalacja graficzna jest niedostępna. Uruchamianie trybu tekstowego."

#: anaconda:627
#, c-format
msgid "Please ssh install@%s to begin the install."
msgstr "Proszę wykonać polecenie ssh install@%s, aby rozpocząć instalację."

#: anaconda:629
msgid "Please ssh install@<host> to continue installation."
msgstr ""
"Proszę wykonać polecenie ssh install@<komputer>, aby kontynuować instalację."

#: pyanaconda/bootloader.py:239
msgid "/boot filesystem"
msgstr "system plików /boot"

#: pyanaconda/bootloader.py:388
#, python-format
msgid ""
"RAID sets that contain '%s' must have one of the following raid levels: %s."
msgstr ""
"Zestawy RAID zawierające \"%s\" muszą być jednego z następujących poziomów "
"RAID: %s."

#: pyanaconda/bootloader.py:396
#, python-format
msgid ""
"RAID sets that contain '%s' must have one of the following metadata "
"versions: %s."
msgstr ""
"Zestawy RAID zawierające \"%s\" muszą posiadać jedną z poniższych wersji "
"metadanych: %s."

#: pyanaconda/bootloader.py:404
#, python-format
msgid ""
"RAID sets that contain '%s' must have one of the following device types: %s."
msgstr ""
"Zestawy RAID zawierające \"%s\" muszą posiadać jeden z poniższych typów "
"urządzeń: %s."

#: pyanaconda/bootloader.py:420
#, python-format
msgid "%s must have one of the following disklabel types: %s."
msgstr "%s musi posiadać jeden z poniższych typów etykiet dysków: %s."

#: pyanaconda/bootloader.py:432
#, python-format
msgid "%s cannot be of type %s."
msgstr "%s nie może być typu %s."

#: pyanaconda/bootloader.py:438
#, python-format
msgid "%s must be mounted on one of %s."
msgstr "%s musi być zamontowane na jednym z %s."

#: pyanaconda/bootloader.py:450
#, python-format
msgid "%s must be between %d and %d MB in size"
msgstr "%s musi mieć rozmiar między %d a %d MB"

#: pyanaconda/bootloader.py:455
#, python-format
msgid "%s must not be smaller than %dMB."
msgstr "%s nie może być mniejsze niż %d MB."

#: pyanaconda/bootloader.py:464
#, python-format
msgid "%s must not be larger than %dMB."
msgstr "%s nie może być większe niż %d MB."

#: pyanaconda/bootloader.py:482
#, python-format
msgid "%s must be within the first %dMB of the disk."
msgstr "%s musi znajdować się w pierwszych %d MB dysku."

#: pyanaconda/bootloader.py:492
#, python-format
msgid "%s must be on a primary partition."
msgstr "%s musi być partycją podstawową."

#: pyanaconda/bootloader.py:616 pyanaconda/bootloader.py:706
#, python-format
msgid "%s cannot be on an encrypted block device."
msgstr "%s nie może znajdować się na zaszyfrowanym urządzeniu blokowym."

#: pyanaconda/bootloader.py:664
#, python-format
msgid "%s cannot be of type %s"
msgstr "%s nie może być typu %s"

#: pyanaconda/bootloader.py:1585
#, python-format
msgid ""
"%s may not have enough space for grub2 to embed core.img when using the %s "
"filesystem on %s"
msgstr ""
"%s może nie posiadać wystarczającej ilości miejsca, aby GRUB2 mógł osadzić "
"core.img podczas używania systemu plików %s na %s"

#: pyanaconda/cmdline.py:66
msgid "Completed"
msgstr "Ukończono"

#: pyanaconda/cmdline.py:74
msgid "In progress"
msgstr "W trakcie wykonywania"

#: pyanaconda/cmdline.py:87
#, python-format
msgid ""
"(%s)\n"
"Command line mode requires all choices to be specified in a kickstart "
"configuration file."
msgstr ""
"(%s)\n"
"Tryb wiersza poleceń wymaga podania wszystkich wyborów w pliku konfiguracji "
"kickstart."

#: pyanaconda/cmdline.py:104
#, python-format
msgid ""
"The following error was found while parsing the kickstart configuration "
"file:\n"
"\n"
"%s"
msgstr ""
"Wystąpił następujący błąd podczas przetwarzania pliku konfiguracji "
"kickstart:\n"
"\n"
"%s"

#: pyanaconda/cmdline.py:113
#, python-format
msgid ""
"(%s)\n"
"%s"
msgstr ""
"(%s)\n"
"%s"

#: pyanaconda/cmdline.py:126 pyanaconda/cmdline.py:131
#: pyanaconda/cmdline.py:137 pyanaconda/cmdline.py:142
#: pyanaconda/ui/tui/hubs/summary.py:58
msgid "Can't have a question in command line mode!"
msgstr "Nie można korzystać ze znaku zapytania w trybie wiersza poleceń."

#: pyanaconda/cmdline.py:161
#, python-format
msgid "In interactive step can't continue. (%s)"
msgstr "Nie można kontynuować w kroku interaktywnym. (%s)"

#: pyanaconda/constants.py:57
msgid ""
"An unhandled exception has occurred.  This is most likely a bug.  Please "
"save a copy of the detailed exception and file a bug report"
msgstr ""
"Wystąpił nieobsługiwany wyjątek. Prawdopodobnie jest to błąd. Proszę zapisać "
"kopię szczegółowego wyjątku i zgłosić raport błędu"

#: pyanaconda/constants.py:63
msgid " with the provider of this software."
msgstr " dostawcy tego oprogramowania."

#: pyanaconda/constants.py:67
#, python-format
msgid " against anaconda at %s"
msgstr " w programie anaconda na %s"

#: pyanaconda/constants.py:95
msgid "Start VNC"
msgstr "Uruchom VNC"

#: pyanaconda/constants.py:96
msgid "Use text mode"
msgstr "Użyj trybu tekstowego"

#: pyanaconda/install.py:59
msgid "Configuring installed system"
msgstr "Konfigurowanie zainstalowanego systemu"

#: pyanaconda/install.py:71
msgid "Writing network configuration"
msgstr "Zapisywanie konfiguracji sieci"

#: pyanaconda/install.py:75
msgid "Creating users"
msgstr "Tworzenie użytkowników"

#: pyanaconda/install.py:82
msgid "Running post install scripts"
msgstr "Wykonywanie skryptów poinstalacyjnych"

#: pyanaconda/install.py:130 pyanaconda/packaging/livepayload.py:133
#: pyanaconda/packaging/yumpayload.py:1631
msgid "Performing post-install setup tasks"
msgstr "Wykonywanie ustawień poinstalacyjnych"

#: pyanaconda/install.py:134
msgid "Installing bootloader"
msgstr "Instalowanie programu startowego"

#: pyanaconda/installclass.py:68
msgid "Install on System"
msgstr "Instalacja w systemie"

#: pyanaconda/installinterfacebase.py:51
msgid "Error Setting Up Repository"
msgstr "Błąd podczas ustawiania repozytorium"

#: pyanaconda/installinterfacebase.py:52
#, python-format
msgid ""
"The following error occurred while setting up the installation repository:\n"
"\n"
"%(e)s\n"
"\n"
"Installation can not continue."
msgstr ""
"Wystąpił poniższy błąd podczas ustawiania repozytorium instalacji:\n"
"\n"
"%(e)s\n"
"\n"
"Instalacja nie może być kontynuowana."

#: pyanaconda/installinterfacebase.py:58
msgid "Exit installer"
msgstr "Zakończ instalator"

#: pyanaconda/installinterfacebase.py:62
#, python-format
msgid ""
"The installation was stopped due to what seems to be a problem with your "
"hardware. The exact error message is:\n"
"\n"
"%s.\n"
"\n"
" The installer will now terminate."
msgstr ""
"Instalacja została zatrzymana z powodu błędu sprzętu. Komunikat błędu:\n"
"\n"
"%s.\n"
"\n"
" Instalator zostanie zakończony."

#: pyanaconda/installinterfacebase.py:65
msgid "Hardware Error Encountered"
msgstr "Wystąpił błąd sprzętu"

#: pyanaconda/installinterfacebase.py:69 pyanaconda/yuminstall.py:263
#: pyanaconda/yuminstall.py:409 pyanaconda/yuminstall.py:835
#: pyanaconda/yuminstall.py:1013 pyanaconda/yuminstall.py:1018
#: pyanaconda/yuminstall.py:1055 pyanaconda/yuminstall.py:1107
#: pyanaconda/yuminstall.py:1113 pyanaconda/yuminstall.py:1269
#: pyanaconda/yuminstall.py:1284 pyanaconda/yuminstall.py:1339
#: pyanaconda/yuminstall.py:1534 pyanaconda/yuminstall.py:1567
#: pyanaconda/yuminstall.py:1590
msgid "_Exit installer"
msgstr "_Zakończ instalator"

#: pyanaconda/iutil.py:898
msgid "the device containing /boot"
msgstr "urządzenie zawierające /boot"

#: pyanaconda/iutil.py:900
#, python-format
msgid ""
"After shutdown, please perform a manual IPL from %s to continue installation."
msgstr ""
"Po wyłączeniu proszę ręcznie wykonać IPL z %s, aby kontynuować instalację."

#: pyanaconda/kickstart.py:155
#, python-format
msgid "Escrow certificate %s requires the network."
msgstr "Certyfikat escrow %s wymaga sieci."

#: pyanaconda/kickstart.py:163
#, python-format
msgid ""
"The following error was encountered while downloading the escrow "
"certificate:\n"
"\n"
"%s"
msgstr ""
"Wystąpił następujący błąd podczas pobierania certyfikatu escrow:\n"
"\n"
"%s"

#: pyanaconda/kickstart.py:482
#, python-format
msgid "%s is missing. Cannot setup firewall."
msgstr "Brak %s. Nie można ustawić zapory sieciowej."

#: pyanaconda/kickstart.py:1466
msgid "Running pre-installation scripts"
msgstr "Wykonywanie skryptów przedinstalacyjnych"

#: pyanaconda/network.py:94
msgid "Hostname cannot be None or an empty string."
msgstr "Nazwa komputera nie może wynosić wartość None lub być pustym ciągiem."

#: pyanaconda/network.py:97
msgid "Hostname must be 255 or fewer characters in length."
msgstr "Nazwa komputera musi składać się z 255 znaków lub mniej."

#: pyanaconda/network.py:103
msgid ""
"Hostname must start with a valid character in the ranges 'a-z', 'A-Z', or "
"'0-9'"
msgstr ""
"Nazwa komputera musi rozpoczynać się od prawidłowego znaku z zakresu \"a-z"
"\", \"A-Z\" lub \"0-9\""

#: pyanaconda/network.py:111
msgid ""
"Hostnames can only contain the characters 'a-z', 'A-Z', '0-9', '-', or '.', "
"parts between periods must contain something and cannot start or end with "
"'-'."
msgstr ""
"Nazwa komputera może zawierać tylko znaki \"a-z\", \"A-Z\", \"0-9\", \"-\" "
"lub \".\", części między kropkami muszą coś zawierać i nie może się zaczynać "
"lub kończyć znakiem \"-\"."

#: pyanaconda/network.py:168
msgid "IP address is missing."
msgstr "Brak adresu IP."

#: pyanaconda/network.py:172
msgid ""
"IPv4 addresses must contain four numbers between 0 and 255, separated by "
"periods."
msgstr ""
"Adresy IPv4 muszą zawierać cztery liczby od 0 do 255 oddzielone kropkami."

#: pyanaconda/network.py:175
#, python-format
msgid "'%s' is not a valid IPv6 address."
msgstr "\"%s\" nie jest prawidłowym adresem IPv6."

#: pyanaconda/network.py:177
#, python-format
msgid "'%s' is an invalid IP address."
msgstr "\"%s\" nie jest prawidłowym adresem IP."

#: pyanaconda/platform.py:58
msgid "bootloader device"
msgstr "urządzenie programu startowego"

#: pyanaconda/platform.py:59
msgid "RAID Device"
msgstr "Urządzenie RAID"

#: pyanaconda/platform.py:60
msgid "First sector of boot partition"
msgstr "Pierwszy sektor partycji startowej"

#: pyanaconda/platform.py:158 pyanaconda/platform.py:341
msgid "Master Boot Record"
msgstr "Główny sektor startowy"

#: pyanaconda/platform.py:194
msgid "EFI System Partition"
msgstr "Partycja systemowa EFI"

#: pyanaconda/platform.py:223
msgid "Apple EFI Boot Partition"
msgstr "Partycja startowa Apple EFI"

#: pyanaconda/platform.py:248
msgid "PReP Boot Partition"
msgstr "Partycja startowa PReP"

#: pyanaconda/platform.py:271
msgid "Apple Bootstrap Partition"
msgstr "Partycja startowa Apple"

#: pyanaconda/platform.py:300
msgid "DASD"
msgstr "DASD"

#: pyanaconda/platform.py:301
msgid "zFCP"
msgstr "zFCP"

#: pyanaconda/platform.py:366
msgid "U-Boot Partition"
msgstr "Partycja U-Boot"

#: pyanaconda/product.py:54
#, python-format
msgid "%(productName)s %(productVersion)s INSTALLATION"
msgstr "INSTALACJA SYSTEMU %(productName)s %(productVersion)s"

#: pyanaconda/rescue.py:171 pyanaconda/rescue.py:290 pyanaconda/rescue.py:375
#, python-format
msgid "Run %s to unmount the system when you are finished."
msgstr "Należy wykonać polecenie %s, aby odmontować system po ukończeniu."

#: pyanaconda/rescue.py:174
msgid "When finished please exit from the shell and your system will reboot."
msgstr ""
"Po ukończeniu należy wyjść z powłoki, a system zostanie ponownie uruchomiony."

#: pyanaconda/rescue.py:188
msgid "Unable to find /bin/sh to execute!  Not starting shell"
msgstr ""
"Nie można odnaleźć /bin/sh do wykonania. Powłoka nie zostanie uruchomiona"

#: pyanaconda/rescue.py:224 pyanaconda/rescue.py:297 pyanaconda/rescue.py:307
#: pyanaconda/rescue.py:378 pyanaconda/rescue.py:409
msgid "Rescue"
msgstr "Ratowanie systemu"

#: pyanaconda/rescue.py:225
#, python-format
msgid ""
"The rescue environment will now attempt to find your Linux installation and "
"mount it under the directory %s.  You can then make any changes required to "
"your system.  If you want to proceed with this step choose 'Continue'.  You "
"can also choose to mount your file systems read-only instead of read-write "
"by choosing 'Read-Only'.  \n"
"\n"
"If for some reason this process fails you can choose 'Skip' and this step "
"will be skipped and you will go directly to a command shell.\n"
"\n"
msgstr ""
"Środowisko ratownicze spróbuje teraz odnaleźć instalację systemu Linux i "
"zamontować ją w katalogu %s. Wtedy będzie można dokonać wszystkich "
"niezbędnych zmian w systemie. Aby kontynuować, należy wybrać przycisk "
"\"Kontynuuj\". Można także zamontować systemy plików w trybie tylko do "
"odczytu, zamiast w trybie pełnego dostępu, wybierając \"Tylko odczyt\".  \n"
"\n"
"Jeśli z jakiegoś powodu ten proces się nie powiedzie, należy wybrać przycisk "
"\"Pomiń\", a krok ten zostanie pominięty i nastąpi przejście od razu do "
"powłoki poleceń.\n"

#: pyanaconda/rescue.py:235 pyanaconda/storage/devicetree.py:96
msgid "Continue"
msgstr "Kontynuuj"

#: pyanaconda/rescue.py:235 pyanaconda/rescue.py:241
msgid "Read-Only"
msgstr "Tylko odczyt"

#: pyanaconda/rescue.py:235 pyanaconda/rescue.py:237
msgid "Skip"
msgstr "Pomiń"

#: pyanaconda/rescue.py:265
msgid "System to Rescue"
msgstr "System do ratowania"

#: pyanaconda/rescue.py:266
msgid "Which device holds the root partition of your installation?"
msgstr "Które urządzenie zawiera partycję root instalacji?"

#: pyanaconda/rescue.py:268 pyanaconda/rescue.py:272
msgid "Exit"
msgstr "Zakończ"

#: pyanaconda/rescue.py:287 pyanaconda/rescue.py:372
msgid "The system will reboot automatically when you exit from the shell."
msgstr ""
"Po wyjściu z powłoki system zostanie automatycznie uruchomiony ponownie."

#: pyanaconda/rescue.py:298
#, python-format
msgid ""
"Your system had dirty file systems which you chose not to mount.  Press "
"return to get a shell from which you can fsck and mount your partitions. %s"
msgstr ""
"System zawiera nieczysto zamknięte systemy plików, których nie zamontowano. "
"Proszę nacisnąć klawisz return, aby uzyskać powłokę, z której będzie można "
"uruchomić fsck i zamontować partycje. %s"

#: pyanaconda/rescue.py:308
#, python-format
msgid ""
"Your system has been mounted under %(rootPath)s.\n"
"\n"
"Press <return> to get a shell. If you would like to make your system the "
"root environment, run the command:\n"
"\n"
"\tchroot %(rootPath)s\n"
"\n"
"%(msg)s"
msgstr ""
"System został zamontowany w katalogu %(rootPath)s.\n"
"\n"
"Naciśnięcie klawisza <return> spowoduje przejście do powłoki. Aby system był "
"środowiskiem roota, należy wykonać polecenie:\n"
"\n"
"\tchroot %(rootPath)s\n"
"\n"
"%(msg)s"

#: pyanaconda/rescue.py:379
#, python-format
msgid ""
"An error occurred trying to mount some or all of your system. Some of it may "
"be mounted under %s.\n"
"\n"
"Press <return> to get a shell."
msgstr ""
"Wystąpił błąd podczas próby montowania części lub całości systemu. Część "
"systemu może być zamontowana jako %s.\n"
"\n"
"Naciśnięcie klawisza <return> spowoduje przejście do powłoki."

#: pyanaconda/rescue.py:387
msgid "You don't have any Linux partitions.  Rebooting.\n"
msgstr "Brak partycji systemu Linux. Ponowne uruchamianie.\n"

#: pyanaconda/rescue.py:391
msgid " The system will reboot automatically when you exit from the shell."
msgstr ""
" Po wyjściu z powłoki system zostanie automatycznie uruchomiony ponownie."

#: pyanaconda/rescue.py:395
msgid "Rescue Mode"
msgstr "Tryb ratunkowy"

#: pyanaconda/rescue.py:396
#, python-format
msgid "You don't have any Linux partitions. Press return to get a shell.%s"
msgstr ""
"Brak partycji systemu Linux. Naciśnięcie klawisza return spowoduje przejście "
"do powłoki.%s"

#: pyanaconda/rescue.py:408
#, python-format
msgid "Your system is mounted under the %s directory."
msgstr "System został zamontowany w katalogu %s."

#: pyanaconda/vnc.py:61
#, python-format
msgid "%(productName)s %(productVersion)s installation"
msgstr "Instalacja systemu %(productName)s %(productVersion)s"

#: pyanaconda/vnc.py:118
#, python-format
msgid "%(productName)s %(productVersion)s installation on host %(name)s"
msgstr ""
"Instalacja systemu %(productName)s %(productVersion)s na komputerze %(name)s"

#: pyanaconda/vnc.py:137
#, python-format
msgid "Attempting to connect to vnc client on host %s..."
msgstr "Próba połączenia się z klientem VNC na komputerze %s..."

#: pyanaconda/vnc.py:151
msgid "Connected!"
msgstr "Połączono."

#: pyanaconda/vnc.py:154
msgid "Will try to connect again in 15 seconds..."
msgstr "Kolejna próba połączenia nastąpi za 15 sekund..."

#: pyanaconda/vnc.py:160
#, python-format
msgid "Giving up attempting to connect after %d try!\n"
msgid_plural "Giving up attempting to connect after %d tries!\n"
msgstr[0] "Po %d próbie połączenie zostanie porzucone.\n"
msgstr[1] "Po %d próbach połączenie zostanie porzucone.\n"
msgstr[2] "Po %d próbach połączenie zostanie porzucone.\n"

#: pyanaconda/vnc.py:171
#, python-format
msgid "Please manually connect your vnc client to %s to begin the install."
msgstr ""
"Proszę ręcznie połączyć się klientem VNC do %s, aby rozpocząć instalację."

#: pyanaconda/vnc.py:173
#, python-format
msgid ""
"Please manually connect your vnc client to <IP ADDRESS>:%s to begin the "
"install. Switch to the shell (Ctrl-B 2) and run 'ip addr' to find the <IP "
"ADDRESS>."
msgstr ""
"Proszę ręcznie połączyć klienta VNC z <ADRES IP>:%s, aby rozpocząć "
"instalację. Należy przełączyć do powłoki (Ctrl-B 2) i wykonać polecenie \"ip "
"addr\", aby odnaleźć <ADRES IP>."

#: pyanaconda/vnc.py:178
msgid "Starting VNC..."
msgstr "Uruchamianie VNC..."

#: pyanaconda/vnc.py:220
msgid "The VNC server is now running."
msgstr "Serwer VNC jest uruchomiony."

#: pyanaconda/vnc.py:224
msgid ""
"\n"
"\n"
"You chose to connect to a listening vncviewer. \n"
"This does not require a password to be set.  If you \n"
"set a password, it will be used in case the connection \n"
"to the vncviewer is unsuccessful\n"
"\n"
msgstr ""
"\n"
"\n"
"Wybrano połączenie z nasłuchującym programem vncviewer. \n"
"Nie wymaga to ustawienia hasła. Jeśli hasło zostanie \n"
"ustawione, zostanie użyte w przypadku, kiedy połączenie \n"
"z programem vncviewer nie powiedzie się\n"
"\n"

#: pyanaconda/vnc.py:229
msgid ""
"\n"
"\n"
"WARNING!!! VNC server running with NO PASSWORD!\n"
"You can use the vncpassword=<password> boot option\n"
"if you would like to secure the server.\n"
"\n"
msgstr ""
"\n"
"\n"
"UWAGA! Serwer VNC jest uruchomiony BEZ HASŁA.\n"
"Można użyć opcji startowej vncpassword=<hasło>,\n"
"aby zabezpieczyć serwer.\n"
"\n"

#: pyanaconda/vnc.py:233
msgid ""
"\n"
"\n"
"You chose to execute vnc with a password. \n"
"\n"
msgstr ""
"\n"
"\n"
"Wybrano wykonanie VNC z hasłem. \n"
"\n"

#: pyanaconda/vnc.py:235
msgid ""
"\n"
"\n"
"Unknown Error.  Aborting. \n"
"\n"
msgstr ""
"\n"
"\n"
"Nieznany błąd. Przerywanie. \n"
"\n"

#: pyanaconda/vnc.py:255
msgid ""
"VNC password provided was not at least 6 characters long.\n"
"Please enter a new one.  Leave blank for no password."
msgstr ""
"Podane hasło VNC nie ma mniej niż 6 znaków.\n"
"Proszę podać nowe. Pozostawienie pustego pola oznacza brak hasła."

#: pyanaconda/yuminstall.py:87
#, python-format
msgid "%s MB"
msgstr "%s MB"

#: pyanaconda/yuminstall.py:90
#, python-format
msgid "%s KB"
msgstr "%s KB"

#: pyanaconda/yuminstall.py:92
#, python-format
msgid "%s Byte"
msgid_plural "%s Bytes"
msgstr[0] "%s bajt"
msgstr[1] "%s bajty"
msgstr[2] "%s bajtów"

#: pyanaconda/yuminstall.py:134 pyanaconda/ui/gui/hubs/progress.glade:83
msgid "Preparing to install"
msgstr "Przygotowywanie do instalacji"

#: pyanaconda/yuminstall.py:135 pyanaconda/packaging/yumpayload.py:1536
msgid "Preparing transaction from installation source"
msgstr "Przygotowywanie transakcji ze źródła instalacji"

#: pyanaconda/yuminstall.py:170
#, python-format
msgid "<b>Upgrading %(pkgStr)s</b> (%(size)s)\n"
msgstr "<b>Aktualizowanie %(pkgStr)s</b> (%(size)s)\n"

#: pyanaconda/yuminstall.py:173
#, python-format
msgid "<b>Installing %(pkgStr)s</b> (%(size)s)\n"
msgstr "<b>Instalowanie %(pkgStr)s</b> (%(size)s)\n"

#: pyanaconda/yuminstall.py:218
#, python-format
msgid "Packages completed: %(donepkgs)d of %(numpkgs)d"
msgid_plural "Packages completed: %(donepkgs)d of %(numpkgs)d"
msgstr[0] "Ukończone pakiety: %(donepkgs)d z %(numpkgs)d"
msgstr[1] "Ukończone pakiety: %(donepkgs)d z %(numpkgs)d"
msgstr[2] "Ukończone pakiety: %(donepkgs)d z %(numpkgs)d"

#: pyanaconda/yuminstall.py:232
#, python-format
msgid "<b>Cleaning up %s</b>"
msgstr "<b>Czyszczenie %s</b>"

#: pyanaconda/yuminstall.py:252
msgid "cpio"
msgstr "cpio"

#: pyanaconda/yuminstall.py:254
msgid "unpack"
msgstr "rozpakowania"

#: pyanaconda/yuminstall.py:256
msgid "script"
msgstr "skryptu"

#: pyanaconda/yuminstall.py:257
msgid "Error Installing Package"
msgstr "Błąd podczas instalowania pakietu"

#: pyanaconda/yuminstall.py:258
#, python-format
msgid ""
"A %s error occurred when installing the %s package.  This could indicate "
"errors when reading the installation media.  Installation cannot continue."
msgstr ""
"Wystąpił błąd %s podczas instalowania pakietu %s. Może to wskazywać na błędy "
"podczas odczytywania nośnika instalacji. Nie można kontynuować instalacji."

#: pyanaconda/yuminstall.py:406
msgid "Wrong Disc"
msgstr "Błędna płyta"

#: pyanaconda/yuminstall.py:407
#, python-format
msgid "That's not the correct %s disc."
msgstr "To nie jest poprawna płyta systemu %s."

#: pyanaconda/yuminstall.py:564
#, python-format
msgid "Repository %r is missing name in configuration, using id"
msgstr ""
"Repozytorium %r nie posiada nazwy w konfiguracji, używanie identyfikatora"

#: pyanaconda/yuminstall.py:830 pyanaconda/yuminstall.py:1264
msgid "No Network Available"
msgstr "Brak dostępnych sieci"

#: pyanaconda/yuminstall.py:831 pyanaconda/yuminstall.py:1265
msgid ""
"Some of your software repositories require networking, but there was an "
"error enabling the network on your system."
msgstr ""
"Niektóre z repozytoriów oprogramowania wymagają sieci, ale wystąpił błąd "
"podczas włączania jej w systemie."

#: pyanaconda/yuminstall.py:919 pyanaconda/yuminstall.py:921
msgid "Re_boot"
msgstr "U_ruchom ponownie"

#: pyanaconda/yuminstall.py:919
msgid "_Eject"
msgstr "_Wysuń"

#: pyanaconda/yuminstall.py:921 pyanaconda/yuminstall.py:1284
#: pyanaconda/yuminstall.py:1339 pyanaconda/yuminstall.py:1567
msgid "_Retry"
msgstr "_Ponów"

#: pyanaconda/yuminstall.py:924 pyanaconda/yuminstall.py:1011
#: pyanaconda/yuminstall.py:1016 pyanaconda/yuminstall.py:1289
#: pyanaconda/yuminstall.py:1349 pyanaconda/yuminstall.py:1561
#: pyanaconda/yuminstall.py:1583 pyanaconda/storage/zfcp.py:375
#: pyanaconda/ui/tui/__init__.py:38
msgid "Error"
msgstr "Błąd"

#: pyanaconda/yuminstall.py:925
#, python-format
msgid ""
"The file %s cannot be opened.  This is due to a missing file, a corrupt "
"package or corrupt media.  Please verify your installation source.\n"
"\n"
"If you exit, your system will be left in an inconsistent state that will "
"likely require reinstallation.\n"
"\n"
msgstr ""
"Nie można otworzyć pliku %s. Powodem może być brak pliku albo uszkodzony "
"pakiet lub nośnik. Proszę sprawdzić źródło instalacji.\n"
"\n"
"Jeśli instalator zostanie zakończony, system pozostanie niespójny, co "
"prawdopodobnie będzie wymagało ponownej instalacji.\n"
"\n"

#: pyanaconda/yuminstall.py:972
msgid "Retrying"
msgstr "Ponowna próba"

#: pyanaconda/yuminstall.py:972
msgid "Retrying download."
msgstr "Ponowna próba pobrania."

#: pyanaconda/yuminstall.py:1007
#, python-format
msgid ""
"There was an error running your transaction for the following reason: %s\n"
msgstr "Wystąpił błąd podczas wykonywania transakcji z powodu: %s\n"

#: pyanaconda/yuminstall.py:1018 pyanaconda/yuminstall.py:1113
#: pyanaconda/yuminstall.py:1536 pyanaconda/yuminstall.py:1590
msgid "_Back"
msgstr "_Wstecz"

#: pyanaconda/yuminstall.py:1051
#, python-format
msgid ""
"There was an error running your transaction for the following reason: %s.\n"
msgstr ""
"Wystąpił błąd podczas wykonywania transakcji z następującego powodu: %s.\n"

#: pyanaconda/yuminstall.py:1053 pyanaconda/yuminstall.py:1105
#: pyanaconda/yuminstall.py:1110
msgid "Error Running Transaction"
msgstr "Błąd podczas wykonywania transakcji"

#: pyanaconda/yuminstall.py:1059 pyanaconda/yuminstall.py:1060
msgid "file conflicts"
msgstr "konflikty plików"

#: pyanaconda/yuminstall.py:1061
msgid "older package(s)"
msgstr "starsze pakiety"

#: pyanaconda/yuminstall.py:1062
msgid "insufficient disk space"
msgstr "niewystarczająca ilość miejsca na dysku"

#: pyanaconda/yuminstall.py:1063
msgid "insufficient disk inodes"
msgstr "niewystarczająca ilość i-węzłów na dysku"

#: pyanaconda/yuminstall.py:1064
msgid "package conflicts"
msgstr "konflikty pakietów"

#: pyanaconda/yuminstall.py:1065
msgid "package already installed"
msgstr "pakiet jest już zainstalowany"

#: pyanaconda/yuminstall.py:1066
msgid "required package"
msgstr "wymagany pakiet"

#: pyanaconda/yuminstall.py:1067
msgid "package for incorrect arch"
msgstr "pakiet dla niepoprawnej architektury"

#: pyanaconda/yuminstall.py:1068
msgid "package for incorrect os"
msgstr "pakiet dla niepoprawnego systemu operacyjnego"

#: pyanaconda/yuminstall.py:1082 pyanaconda/packaging/yumpayload.py:1364
msgid "You need more space on the following file systems:\n"
msgstr "Wymagane jest więcej miejsca na następujących systemach plików:\n"

#: pyanaconda/yuminstall.py:1095
#, python-format
msgid ""
"There were file conflicts when checking the packages to be installed:\n"
"%s\n"
msgstr ""
"Wystąpił konflikt plików podczas sprawdzania instalowanych pakietów:\n"
"%s\n"

#: pyanaconda/yuminstall.py:1098
#, python-format
msgid ""
"There was an error running your transaction for the following reason(s): "
"%s.\n"
msgstr ""
"Wystąpił błąd podczas wykonywania transakcji z następujących powodów: %s.\n"

#: pyanaconda/yuminstall.py:1290
msgid ""
"Unable to read group information from repositories.  This is a problem with "
"the generation of your install tree."
msgstr ""
"Nie można odczytać informacji o grupach z repozytoriów. Jest to problem z "
"tworzeniem drzewa instalacji."

#: pyanaconda/yuminstall.py:1339
msgid "Edit"
msgstr "Zmodyfikuj"

#: pyanaconda/yuminstall.py:1344 pyanaconda/yuminstall.py:1534
#: pyanaconda/ui/gui/spokes/storage.glade:43
#: pyanaconda/ui/gui/spokes/storage.glade:952
#: widgets/src/StandaloneWindow.c:53
msgid "_Continue"
msgstr "_Kontynuuj"

#: pyanaconda/yuminstall.py:1350 pyanaconda/yuminstall.py:1562
#, python-format
msgid ""
"Unable to read package metadata. This may be due to a missing repodata "
"directory.  Please ensure that your install tree has been correctly "
"generated.\n"
"\n"
"%s"
msgstr ""
"Nie można odczytać metadanych pakietu. Powodem może być brak katalogu "
"repodata. Proszę upewnić się, że drzewo instalacji zostało poprawnie "
"utworzone.\n"
"\n"
"%s"

#: pyanaconda/yuminstall.py:1538
msgid "Warning"
msgstr "Ostrzeżenie"

#: pyanaconda/yuminstall.py:1539
msgid ""
"Some of the packages you have selected for install are missing "
"dependencies.  You can exit the installation, go back and change your "
"package selections, or continue installing these packages without their "
"dependencies.  If you continue, these packages may not work correctly due to "
"missing components."
msgstr ""
"Niektóre z pakietów wybranych do instalacji posiadają brakujące zależności. "
"Można zakończyć instalację, powrócić i zmienić wybór pakietów lub "
"kontynuować instalowanie tych pakietów bez ich zależności. Jeśli instalacja "
"zostanie kontynuowana, te pakiety mogą nie działać poprawnie z powodu braku "
"ich składników."

#: pyanaconda/yuminstall.py:1584
#, python-format
msgid ""
"Your selected packages require %d MB of free space for installation, but you "
"do not have enough available.  You can change your selections or exit the "
"installer."
msgstr ""
"Wybrane pakiety wymagają %d MB wolnego miejsca do instalacji, ale nie jest "
"tyle dostępne. Można zmienić wybór lub zakończyć instalator."

#: pyanaconda/yuminstall.py:1669
msgid "Post Upgrade"
msgstr "Po aktualizacji"

#: pyanaconda/yuminstall.py:1670
msgid "Performing post-upgrade configuration"
msgstr "Wykonywanie konfiguracji systemu po aktualizacji"

#: pyanaconda/yuminstall.py:1672
msgid "Post Installation"
msgstr "Po instalacji"

#: pyanaconda/yuminstall.py:1673
msgid "Performing post-installation configuration"
msgstr "Wykonywanie konfiguracji systemu po instalacji"

#: pyanaconda/yuminstall.py:1832
msgid "Installation Starting"
msgstr "Rozpoczynanie instalacji"

#: pyanaconda/yuminstall.py:1833
msgid "Starting installation process"
msgstr "Rozpoczynanie instalacji"

#: pyanaconda/yuminstall.py:1871
msgid "Dependency Check"
msgstr "Sprawdzanie zależności"

#: pyanaconda/yuminstall.py:1872
msgid "Checking dependencies in packages selected for installation"
msgstr "Sprawdzanie zależności pakietów wybranych do instalacji"

#: pyanaconda/yuminstall.py:1928 pyanaconda/yuminstall.py:1944
msgid "Retrieving installation information."
msgstr "Pobieranie informacji o instalacji."

#: pyanaconda/yuminstall.py:1930 pyanaconda/yuminstall.py:1946
#, python-format
msgid "Retrieving installation information for %s."
msgstr "Pobieranie informacji o instalacji dla %s."

#: pyanaconda/yuminstall.py:1931 pyanaconda/yuminstall.py:1947
msgid "Installation Progress"
msgstr "Postęp instalacji"

#: pyanaconda/installclasses/fedora.py:35
msgid "_Fedora"
msgstr "_Fedora"

#: pyanaconda/installclasses/fedora.py:36
#, python-format
msgid ""
"The default installation of %s includes a set of software applicable for "
"general internet usage. You can optionally select a different set of "
"software now."
msgstr ""
"Domyśla instalacja systemu %s zawiera zestaw oprogramowania odpowiedni do "
"ogólnego używania Internetu. Można teraz opcjonalnie wybrać inny zestaw."

#: pyanaconda/installclasses/fedora.py:45
msgid "Graphical Desktop"
msgstr "Środowisko graficzne"

#: pyanaconda/installclasses/fedora.py:50
msgid "Software Development"
msgstr "Tworzenie oprogramowania"

#: pyanaconda/installclasses/fedora.py:56
msgid "Web Server"
msgstr "Serwer WWW"

#: pyanaconda/installclasses/fedora.py:60 pyanaconda/installclasses/rhel.py:39
msgid "Minimal"
msgstr "Minimalna"

#: pyanaconda/installclasses/rhel.py:28
msgid "Red Hat Enterprise Linux"
msgstr "Red Hat Enterprise Linux"

#: pyanaconda/installclasses/rhel.py:29
#, python-format
msgid ""
"The default installation of %s is a minimum install. You can optionally "
"select a different set of software now."
msgstr ""
"Domyśla instalacja systemu %s jest instalacją minimalną. Można teraz "
"opcjonalnie wybrać inny zestaw oprogramowania."

#: pyanaconda/packaging/livepayload.py:72
#: pyanaconda/packaging/livepayload.py:90
msgid "Installing software"
msgstr "Instalowanie oprogramowania"

#: pyanaconda/packaging/yumpayload.py:1300
msgid "Starting package installation process"
msgstr "Rozpoczynanie procesu instalacji pakietów"

#: pyanaconda/packaging/yumpayload.py:1556
msgid "Upgrading"
msgstr "Aktualizowanie"

#: pyanaconda/packaging/yumpayload.py:1558
msgid "Installing"
msgstr "Instalowanie"

#: pyanaconda/storage/__init__.py:974
msgid "This partition is holding the data for the hard drive install."
msgstr "Ta partycja przechowuje dane do instalacji z dysku twardego."

#: pyanaconda/storage/__init__.py:979
msgid "You cannot delete a partition of a LDL formatted DASD."
msgstr ""
"Nie można usunąć partycji będącej elementem urządzenia DASD sformatowanego "
"metodą LDL."

#: pyanaconda/storage/__init__.py:985
#, python-format
msgid "This device is part of the RAID device %s."
msgstr "Te urządzenie jest częścią urządzenia RAID %s."

#: pyanaconda/storage/__init__.py:988
msgid "This device is part of a RAID device."
msgstr "Te urządzenie jest częścią urządzenia RAID."

#: pyanaconda/storage/__init__.py:991
msgid "This device is part of an inconsistent LVM Volume Group."
msgstr "To urządzenie jest częścią niespójnej grupy woluminów LVM."

#: pyanaconda/storage/__init__.py:996
#, python-format
msgid "This device is part of the LVM volume group '%s'."
msgstr "Te urządzenie jest częścią grupy woluminów LVM \"%s\"."

#: pyanaconda/storage/__init__.py:999
msgid "This device is part of a LVM volume group."
msgstr "Te urządzenie jest częścią grupy woluminów LVM."

#: pyanaconda/storage/__init__.py:1015
msgid ""
"This device is an extended partition which contains logical partitions that "
"cannot be deleted:\n"
"\n"
msgstr ""
"Te urządzenie jest rozszerzoną partycją, zawierającą partycji logiczne, "
"których nie można usunąć:\n"
"\n"

#: pyanaconda/storage/__init__.py:1262 pyanaconda/ui/gui/spokes/custom.py:1006
msgid "You must create a new filesystem on the root device."
msgstr "Należy utworzyć nowy system plików na głównym urządzeniu."

#: pyanaconda/storage/__init__.py:1450
#, python-format
msgid ""
"You have not defined a root partition (/), which is required for "
"installation of %s to continue."
msgstr ""
"Nie wybrano głównej partycji (\"/\"), która jest niezbędna do kontynuowania "
"instalacji systemu %s."

#: pyanaconda/storage/__init__.py:1455
#, python-format
msgid ""
"Your root partition is less than 250 megabytes which is usually too small to "
"install %s."
msgstr ""
"Główna partycja ma mniej niż 250 megabajtów, a to zwykle za mało, aby "
"zainstalować system %s."

#: pyanaconda/storage/__init__.py:1469
msgid ""
"This platform requires /boot on a dedicated partition or logical volume.  If "
"you do not want a /boot volume, you must place / on a dedicated non-LVM "
"partition."
msgstr ""
"Ta platforma wymaga /boot na oddzielnej partycji lub woluminie logicznym. "
"Jeśli /boot nie ma znajdować się na własnym woluminie, to należy umieścić / "
"na oddzielnej partycji nie będącej LVM."

#: pyanaconda/storage/__init__.py:1478
#, python-format
msgid ""
"Your %(mount)s partition is less than %(size)s megabytes which is lower than "
"recommended for a normal %(productName)s install."
msgstr ""
"Partycja %(mount)s ma mniej niż %(size)s MB, czyli mniej niż jest zalecane "
"do zwykłej instalacji systemu %(productName)s."

#: pyanaconda/storage/__init__.py:1488
#, python-format
msgid ""
"Your %(mount)s partition is too small for %(format)s formatting (allowable "
"size is %(minSize)d MB to %(maxSize)d MB)"
msgstr ""
"Partycja %(mount)s jest za mała dla formatowania %(format)s (dozwolony jest "
"rozmiar od %(minSize)d MB do %(maxSize)d MB)"

#: pyanaconda/storage/__init__.py:1493
#, python-format
msgid ""
"Your %(mount)s partition is too large for %(format)s formatting (allowable "
"size is %(minSize)d MB to %(maxSize)d MB)"
msgstr ""
"Partycja %(mount)s jest za duża dla formatowania %(format)s (dozwolony jest "
"rozmiar od %(minSize)d MB do %(maxSize)d MB)"

#: pyanaconda/storage/__init__.py:1520
msgid ""
"Installing on a USB device.  This may or may not produce a working system."
msgstr ""
"Instalowanie na urządzeniu USB. W ten sposób można lub nie uzyskać "
"działający system."

#: pyanaconda/storage/__init__.py:1523
msgid ""
"Installing on a FireWire device.  This may or may not produce a working "
"system."
msgstr ""
"Instalowanie na urządzeniu FireWire. W ten sposób można lub nie uzyskać "
"działający system."

#: pyanaconda/storage/__init__.py:1529
msgid "you have not created a bootloader stage1 target device"
msgstr "nie utworzono urządzenia docelowego 1 etapu programu startowego"

#: pyanaconda/storage/__init__.py:1538
msgid "You have not created a bootable partition."
msgstr "Nie utworzono partycji startowej."

#: pyanaconda/storage/__init__.py:1559
#, python-format
msgid ""
"Your BIOS-based system needs a special partition to boot with %s's new disk "
"label format (GPT). To continue, please create a 1MB 'BIOS Boot' type "
"partition."
msgstr ""
"Komputer oparty na BIOS-ie wymaga specjalnej partycji do uruchamiania za "
"pomocą nowego formatu etykiet dysków %s (GPT). Aby kontynuować, proszę "
"utworzyć partycję typu \"Partycja startowa BIOS\" o rozmiarze 1 MB."

#: pyanaconda/storage/__init__.py:1572
#, python-format
msgid ""
"You have not specified a swap partition.  %(requiredMem)s MB of memory is "
"required to continue installation without a swap partition, but you only "
"have %(installedMem)s MB."
msgstr ""
"Nie podano partycji wymiany. Wymagane jest %(requiredMem)s MB pamięci, aby "
"kontynuować instalację bez partycji wymiany, a w komputerze jest tylko "
"%(installedMem)s MB."

#: pyanaconda/storage/__init__.py:1578
msgid ""
"You have not specified a swap partition.  Although not strictly required in "
"all cases, it will significantly improve performance for most installations."
msgstr ""
"Nie podano partycji wymiany. Chociaż nie zawsze jest ona ściśle wymagana, w "
"większości przypadków znacznie poprawia wydajność systemu."

#: pyanaconda/storage/__init__.py:1584
msgid ""
"At least one of your swap devices does not have a UUID, which is common in "
"swap space created using older versions of mkswap. These devices will be "
"referred to by device path in /etc/fstab, which is not ideal since device "
"paths can change under a variety of circumstances. "
msgstr ""
"Co najmniej jedno z urządzeń wymiany nie posiada UUID, co jest częste w "
"przypadku przestrzeni wymiany utworzonych używając starszych wersji programu "
"mkswap. Te urządzenia będą nazywane według ścieżek do nich w pliku /etc/"
"fstab, co nie jest najlepsze, ponieważ ścieżki te mogą się zmieniać w wielu "
"okolicznościach. "

#: pyanaconda/storage/__init__.py:1594
#, python-format
msgid ""
"This mount point is invalid.  The %s directory must be on the / file system."
msgstr ""
"Ten punkt montowania jest nieprawidłowy. Katalog %s musi znajdować się w "
"systemie plików /."

#: pyanaconda/storage/__init__.py:1598
#, python-format
msgid "The mount point %s must be on a linux file system."
msgstr ""
"Punkt montowania %s musi znajdować się na systemie plików systemu Linux."

#: pyanaconda/storage/__init__.py:3199
#, python-format
msgid "Linux on %s"
msgstr "System Linux na %s"

#: pyanaconda/storage/__init__.py:3204
msgid "Unknown Linux"
msgstr "Nieznany system Linux"

#: pyanaconda/storage/__init__.py:3206
#, python-format
msgid "%(product)s Linux %(version)s for %(arch)s"
msgstr "System %(product)s Linux %(version)s dla %(arch)s"

#: pyanaconda/storage/dasd.py:159
msgid "Formatting DASD Device"
msgid_plural "Formatting DASD Devices"
msgstr[0] "Formatowanie urządzenia DASD"
msgstr[1] "Formatowanie urządzeń DASD"
msgstr[2] "Formatowanie urządzeń DASD"

#: pyanaconda/storage/dasd.py:160
#, python-format
msgid "Preparing %d DASD device for use with Linux..."
msgid_plural "Preparing %d DASD devices for use with Linux..."
msgstr[0] "Przygotowywanie %d urządzenia DASD do użycia z systemem Linux..."
msgstr[1] "Przygotowywanie %d urządzeń DASD do użycia z systemem Linux..."
msgstr[2] "Przygotowywanie %d urządzeń DASD do użycia z systemem Linux..."

#: pyanaconda/storage/deviceaction.py:419
#, python-format
msgid "Creating %(type)s on %(device)s"
msgstr "Tworzenie %(type)s na urządzeniu %(device)s"

#: pyanaconda/storage/deviceaction.py:559
#, python-format
msgid "Resizing filesystem on %(device)s"
msgstr "Zmienianie rozmiaru systemu plików na urządzeniu %(device)s"

#: pyanaconda/storage/deviceaction.py:608
#, python-format
msgid "Migrating filesystem on %(device)s"
msgstr "Migrowanie systemu plików na urządzeniu %(device)s"

#: pyanaconda/storage/devicelibs/lvm.py:310
#, python-format
msgid "vginfo failed for %s"
msgstr "wykonanie vginfo dla %s się nie powiodło"

#: pyanaconda/storage/devicelibs/lvm.py:339
#, python-format
msgid "lvs failed for %s"
msgstr "wykonanie lvs dla %s się nie powiodło"

#: pyanaconda/storage/devices.py:2672
#, python-format
msgid "A RAID%(raidLevel)d set requires at least %(minMembers)d member"
msgid_plural "A RAID%(raidLevel)d set requires at least %(minMembers)d members"
msgstr[0] "Zestaw RAID%(raidLevel)d wymaga co najmniej %(minMembers)d element"
msgstr[1] "Zestaw RAID%(raidLevel)d wymaga co najmniej %(minMembers)d elementy"
msgstr[2] ""
"Zestaw RAID%(raidLevel)d wymaga co najmniej %(minMembers)d elementów"

#: pyanaconda/storage/devicetree.py:96
msgid "Back"
msgstr "Wstecz"

#: pyanaconda/storage/devicetree.py:105
msgid "Confirm"
msgstr "Potwierdzenie"

#: pyanaconda/storage/devicetree.py:106
#, python-format
msgid ""
"Are you sure you want to skip entering a passphrase for device %s?\n"
"\n"
"If you skip this step the device's contents will not be available during "
"installation."
msgstr ""
"Na pewno pominąć podawanie hasła dla urządzenia %s?\n"
"\n"
"Jeśli ten krok zostanie pominięty, zawartość urządzenia nie będzie dostępna "
"podczas instalacji."

#: pyanaconda/storage/fcoe.py:110
msgid "FCoE not available"
msgstr "FCoE jest niedostępne"

#: pyanaconda/storage/formats/fs.py:104
msgid "filesystem configuration missing a type"
msgstr "brak typu w konfiguracji systemu plików"

#: pyanaconda/storage/formats/fs.py:483
#, python-format
msgid "Unknown return code: %d."
msgstr "Nieznany kod zwrotny: %d."

#: pyanaconda/storage/formats/fs.py:504
#, python-format
msgid "%(type)s filesystem check failure on %(device)s: "
msgstr ""
"Sprawdzanie systemu plików %(type)s na urządzeniu %(device)s nie powiodło "
"się: "

#: pyanaconda/storage/formats/fs.py:511
msgid ""
"Errors like this usually mean there is a problem with the filesystem that "
"will require user interaction to repair.  Before restarting installation, "
"reboot to rescue mode or another system that allows you to repair the "
"filesystem interactively.  Restart installation after you have corrected the "
"problems on the filesystem."
msgstr ""
"Tego typu błędy zwykle wskazują na problem z systemem plików, który będzie "
"wymagał działania użytkownika w celu naprawienia go. Przed ponownym "
"uruchomieniem instalacji należy uruchomić tryb ratunkowy lub inny system "
"umożliwiający interaktywne naprawienie systemu plików. Po ukończeniu należy "
"ponownie uruchomić instalację."

#: pyanaconda/storage/formats/fs.py:519
msgid "Unrecoverable Error"
msgstr "Nieodwracalny błąd"

#: pyanaconda/storage/formats/fs.py:883
msgid "File system errors left uncorrected."
msgstr "Błędy systemu plików nie zostały naprawione."

#: pyanaconda/storage/formats/fs.py:884
msgid "Operational error."
msgstr "Błąd działania."

#: pyanaconda/storage/formats/fs.py:885
msgid "Usage or syntax error."
msgstr "Błąd użycia lub składni."

#: pyanaconda/storage/formats/fs.py:886
msgid "e2fsck cancelled by user request."
msgstr "e2fsck zostało anulowane na żądanie użytkownika."

#: pyanaconda/storage/formats/fs.py:887
msgid "Shared library error."
msgstr "Błąd biblioteki współdzielonej."

#: pyanaconda/storage/formats/fs.py:1062
msgid ""
"Recoverable errors have been detected or dosfsck has discovered an internal "
"inconsistency."
msgstr ""
"Wykryto odwracalne błędy lub program dosfsck wykrył wewnętrzną niespójność."

#: pyanaconda/storage/formats/fs.py:1064
msgid "Usage error."
msgstr "Błąd użycia."

#: pyanaconda/storage/formats/luks.py:48
msgid "Encrypted"
msgstr "Zaszyfrowane"

#: pyanaconda/storage/iscsi.py:123
msgid "Unable to change iSCSI initiator name once set"
msgstr "Nie można zmienić nazwy inicjatora iSCSI po jej ustawieniu"

#: pyanaconda/storage/iscsi.py:125
msgid "Must provide an iSCSI initiator name"
msgstr "Należy podać nazwę inicjatora iSCSI."

#: pyanaconda/storage/iscsi.py:293
msgid "iSCSI not available"
msgstr "iSCSI jest niedostępne"

#: pyanaconda/storage/iscsi.py:295
msgid "No initiator name set"
msgstr "Nie ustawiono nazwy inicjatora"

#: pyanaconda/storage/iscsi.py:334
msgid "Logging in to iSCSI node"
msgstr "Logowanie do węzła iSCSI"

#: pyanaconda/storage/iscsi.py:335
#, python-format
msgid "Logging in to iSCSI node %s"
msgstr "Logowanie do węzła iSCSI %s"

#: pyanaconda/storage/iscsi.py:369
msgid "No iSCSI nodes discovered"
msgstr "Nie wykryto węzłów iSCSI"

#: pyanaconda/storage/iscsi.py:391
msgid "No new iSCSI nodes discovered"
msgstr "Nie wykryto nowych węzłów iSCSI"

#: pyanaconda/storage/iscsi.py:394
msgid "Could not log in to any of the discovered nodes"
msgstr "Nie można zalogować się do żadnych wykrytych węzłów"

#: pyanaconda/storage/partitioning.py:291
msgid "No usable disks selected"
msgstr "Nie wybrano żadnych używalnych dysków"

#: pyanaconda/storage/partitioning.py:299
msgid "Not enough free space on disks for automatic partitioning"
msgstr ""
"Brak wystarczającej ilości wolnego miejsca na dyskach dla automatycznego "
"partycjonowania"

#: pyanaconda/storage/partitioning.py:629
msgid "unable to allocate aligned partition"
msgstr "nie można przydzielić wyrównanej partycji"

#: pyanaconda/storage/partitioning.py:637
msgid "requested size exceeds maximum allowed"
msgstr "zażądany rozmiar przekracza dozwolone maksimum"

#: pyanaconda/storage/partitioning.py:728
msgid "can't allocate partitions without platform data"
msgstr "nie można przydzielić partycji bez danych platformy"

#: pyanaconda/storage/partitioning.py:739
#, python-format
msgid "disk %s inaccessible"
msgstr "dysk %s jest niedostępny"

#: pyanaconda/storage/partitioning.py:793
#, python-format
msgid ""
"partition is too small for %(format)s formatting (allowable size is "
"%(minSize)d MB to %(maxSize)d MB)"
msgstr ""
"partycja jest za mała dla formatowania %(format)s (dozwolony rozmiar to od "
"%(minSize)d MB do %(maxSize)d MB)"

#: pyanaconda/storage/partitioning.py:798
#, python-format
msgid ""
"partition is too large for %(format)s formatting (allowable size is "
"%(minSize)d MB to %(maxSize)d MB)"
msgstr ""
"partycja jest zaduża dla formatowania %(format)s (dozwolony rozmiar to od "
"%(minSize)d MB do %(maxSize)d MB)"

#: pyanaconda/storage/partitioning.py:1039
msgid "not enough free space on disks"
msgstr "brak wystarczającej ilości wolnego miejsca na dyskach"

#: pyanaconda/storage/partitioning.py:1060
msgid "not enough free space after creating extended partition"
msgstr ""
"brak wystarczającej ilości wolnego miejsca po utworzeniu rozszerzonej "
"partycji"

#: pyanaconda/storage/partitioning.py:1232
msgid "cannot reclaim more than request has grown"
msgstr "nie można odzyskać więcej niż powiększono żądanie"

#: pyanaconda/storage/partitioning.py:1412
msgid "DiskChunk requests must be of type PartitionRequest"
msgstr "Żądania DiskChunk muszą być typu PartitionRequest"

#: pyanaconda/storage/partitioning.py:1425
msgid "partitions allocated outside disklabel limits"
msgstr "partycje są przydzielone poza ograniczeniami etykiet dysków"

#: pyanaconda/storage/partitioning.py:1505
msgid "VGChunk requests must be of type LVRequest"
msgstr "Żądania VGChunk muszą być typu LVRequest"

#: pyanaconda/storage/partitioning.py:1934
msgid "not enough space for LVM requests"
msgstr "brak wystarczającej ilości wolnego miejsca dla żądań LVM"

#: pyanaconda/storage/zfcp.py:51
msgid "You have not specified a device number or the number is invalid"
msgstr "Nie podano numeru urządzenia lub podany numer jest nieprawidłowy"

#: pyanaconda/storage/zfcp.py:53
msgid "You have not specified a worldwide port name or the name is invalid."
msgstr "Nie podano nazwy World Wide Port lub podana nazwa jest nieprawidłowa."

#: pyanaconda/storage/zfcp.py:55
msgid "You have not specified a FCP LUN or the number is invalid."
msgstr "Nie podano FCP LUN lub podany numer jest nieprawidłowy."

#: pyanaconda/storage/zfcp.py:133
#, python-format
msgid "zFCP device %s not found, not even in device ignore list."
msgstr ""
"Nie odnaleziono urządzenia zFCP %s, nawet na liście ignorowanych urządzeń."

#: pyanaconda/storage/zfcp.py:143
#, python-format
msgid "Could not set zFCP device %(devnum)s online (%(e)s)."
msgstr ""
"Błąd: nie można ustawić urządzenia zFCP %(devnum)s w tryb online (%(e)s)."

#: pyanaconda/storage/zfcp.py:154
#, python-format
msgid "Could not add WWPN %(wwpn)s to zFCP device %(devnum)s (%(e)s)."
msgstr ""
"Błąd: nie można dodać WWPN %(wwpn)s do urządzenia zFCP %(devnum)s (%(e)s)."

#: pyanaconda/storage/zfcp.py:161
#, python-format
msgid "WWPN %(wwpn)s not found at zFCP device %(devnum)s."
msgstr "Nie odnaleziono WWPN %(wwpn)s na urządzeniu zFCP %(devnum)s."

#: pyanaconda/storage/zfcp.py:176
#, python-format
msgid ""
"Could not add LUN %(fcplun)s to WWPN %(wwpn)s on zFCP device %(devnum)s "
"(%(e)s)."
msgstr ""
"Błąd: nie można dodać LUN %(fcplun)s do WWPN %(wwpn)s na urządzeniu zFCP "
"%(devnum)s (%(e)s)."

#: pyanaconda/storage/zfcp.py:182
#, python-format
msgid ""
"LUN %(fcplun)s at WWPN %(wwpn)s on zFCP device %(devnum)s already configured."
msgstr ""
"LUN %(fcplun)s w WWPN %(wwpn)s na urządzeniu zFCP %(devnum)s jest już "
"skonfigurowane."

#: pyanaconda/storage/zfcp.py:194
#, python-format
msgid ""
"Could not read failed attribute of LUN %(fcplun)s at WWPN %(wwpn)s on zFCP "
"device %(devnum)s (%(e)s)."
msgstr ""
"Nie można odczytać nieprawidłowego atrybutu LUN %(fcplun)s w WWPN %(wwpn)s "
"na urządzeniu zFCP %(devnum)s (%(e)s)."

#: pyanaconda/storage/zfcp.py:203
#, python-format
msgid ""
"Failed LUN %(fcplun)s at WWPN %(wwpn)s on zFCP device %(devnum)s removed "
"again."
msgstr ""
"Ponowne usunięcie LUN %(fcplun)s w WWPN %(wwpn)s na urządzeniu zFCP "
"%(devnum)s się nie powiodło."

#: pyanaconda/storage/zfcp.py:260
#, python-format
msgid ""
"Could not correctly delete SCSI device of zFCP %(devnum)s %(wwpn)s "
"%(fcplun)s (%(e)s)."
msgstr ""
"Nie można poprawnie usunąć urządzenia SCSI z zFCP %(devnum)s %(wwpn)s "
"%(fcplun)s (%(e)s)."

#: pyanaconda/storage/zfcp.py:269
#, python-format
msgid ""
"Could not remove LUN %(fcplun)s at WWPN %(wwpn)s on zFCP device %(devnum)s "
"(%(e)s)."
msgstr ""
"Błąd: nie można usunąć LUN %(fcplun)s z WWPN %(wwpn)s na urządzeniu zFCP "
"%(devnum)s (%(e)s)."

#: pyanaconda/storage/zfcp.py:287
#, python-format
msgid "Could not remove WWPN %(wwpn)s on zFCP device %(devnum)s (%(e)s)."
msgstr ""
"Błąd: nie można usunąć WWPN %(wwpn)s na urządzeniu zFCP %(devnum)s (%(e)s)."

#: pyanaconda/storage/zfcp.py:313
#, python-format
msgid "Could not set zFCP device %(devnum)s offline (%(e)s)."
msgstr ""
"Błąd: nie można ustawić urządzenia zFCP %(devnum)s w tryb offline (%(e)s)."

#: pyanaconda/ui/lib/space.py:33
#, python-format
msgid ""
"Not enough space in filesystems for the current software selection. An "
"additional %s is needed."
msgstr ""
"Brak wystarczającej ilości miejsca na systemach plików, aby zmieścić bieżący "
"wybór oprogramowania. Wymagane jest dodatkowe %s."

#: pyanaconda/ui/tui/hubs/progress.py:37
msgid "Installation Hub"
msgstr "Główny ekran instalacji"

#: pyanaconda/ui/tui/hubs/progress.py:117
msgid "\tInstallation complete.  Press return to quit"
msgstr "\tUkończono instalację. Naciśnięcie przycisku Return zakończy"

#: pyanaconda/ui/tui/hubs/summary.py:32
msgid "Install hub"
msgstr "Główny ekran instalacji"

#: pyanaconda/ui/tui/hubs/summary.py:38
msgid "Starting automated install"
msgstr "Uruchamianie instalacji automatycznej"

#: pyanaconda/ui/tui/hubs/__init__.py:43
msgid "Default HUB title"
msgstr "Domyślny tytuł ekranu głównego"

#: pyanaconda/ui/tui/hubs/__init__.py:104
#: pyanaconda/ui/tui/simpleline/base.py:301
msgid "c"
msgstr "c"

#: pyanaconda/ui/tui/hubs/__init__.py:107
msgid "Please complete all spokes before continuing"
msgstr "Proszę wypełnić wszystkie ekrany przed kontynuowaniem"

#: pyanaconda/ui/tui/simpleline/base.py:306
msgid "q"
msgstr "q"

#: pyanaconda/ui/tui/simpleline/base.py:308
msgid "Do you really want to quit?"
msgstr "Na pewno zakończyć?"

#: pyanaconda/ui/tui/simpleline/base.py:407
msgid "  Please make your choice from above ['q' to quit | 'c' to continue]: "
msgstr ""
"  Proszę wybrać jedno z powyższych [\"q\" zakończy | \"c\" kontynuuje]: "

#: pyanaconda/ui/tui/spokes/askvnc.py:32
msgid "VNC"
msgstr "VNC"

#: pyanaconda/ui/tui/spokes/askvnc.py:43
msgid ""
"X was unable to start on your machine.  Would you like to start VNC to "
"connect to this computer from another computer and perform a graphical "
"install or continue with a text mode install?"
msgstr ""
"X nie mógł zostać uruchomiony. Uruchomić VNC, aby połączyć się z tym "
"komputerem z innego i wykonać instalację w trybie graficznym, czy "
"kontynuować instalację w trybie tekstowym?"

#: pyanaconda/ui/tui/spokes/askvnc.py:95
msgid "VNC Password"
msgstr "Hasło VNC"

#: pyanaconda/ui/tui/spokes/askvnc.py:104
msgid ""
"Please provide VNC password. You will have to type it twice. \n"
"Leave blank for no password"
msgstr ""
"Proszę podać hasło VNC. Należy je podać dwukrotnie.\n"
"Pozostawienie pustego pola oznacza brak hasła."

#: pyanaconda/ui/tui/spokes/askvnc.py:123
#: pyanaconda/ui/tui/spokes/password.py:64
msgid "Password: "
msgstr "Hasło: "

#: pyanaconda/ui/tui/spokes/askvnc.py:124
#: pyanaconda/ui/tui/spokes/password.py:65
msgid "Password (confirm): "
msgstr "Hasło (potwierdzenie): "

#: pyanaconda/ui/tui/spokes/askvnc.py:127
msgid "Passwords do not match!"
msgstr "Hasła się nie zgadzają."

#: pyanaconda/ui/tui/spokes/askvnc.py:130
msgid "The password must be at least six characters long."
msgstr "Hasło musi zawierać co najmniej sześć znaków."

#: pyanaconda/ui/tui/spokes/password.py:35
msgid "Set root password"
msgstr "Ustaw hasło roota"

#: pyanaconda/ui/tui/spokes/password.py:49
msgid "Password is set."
msgstr "Hasło jest ustawione."

#: pyanaconda/ui/tui/spokes/password.py:51
msgid "Root account is disabled."
msgstr "Konto roota jest wyłączone."

#: pyanaconda/ui/tui/spokes/password.py:53
msgid "Password is not set."
msgstr "Hasło nie jest ustawione."

#: pyanaconda/ui/tui/spokes/password.py:58
msgid "Please select new root password. You will have to type it twice."
msgstr "Proszę wybrać nowe hasło roota. Należy je podać dwukrotnie."

#: pyanaconda/ui/tui/spokes/password.py:76
#, python-format
msgid "You have provided a weak password: %s. "
msgstr "Podano słabe hasło: %s"

#: pyanaconda/ui/tui/spokes/password.py:77
msgid ""
"\n"
"Would you like to use it anyway?"
msgstr ""
"\n"
"Użyć je mimo to?"

#: pyanaconda/ui/tui/spokes/storage.py:47
msgid "Use All Space"
msgstr "Użyj całego miejsca"

#: pyanaconda/ui/tui/spokes/storage.py:48
msgid "Replace Existing Linux system(s)"
msgstr "Zastąp istniejące systemy Linux"

#: pyanaconda/ui/tui/spokes/storage.py:49
msgid "Use Free Space"
msgstr "Użyj wolnego miejsca"

#: pyanaconda/ui/tui/spokes/storage.py:92
msgid "Install Destination"
msgstr "Cel instalacji"

#: pyanaconda/ui/tui/spokes/storage.py:120
#: pyanaconda/ui/gui/spokes/storage.py:439
msgid "No disks selected"
msgstr "Nie wybrano dysku"

#: pyanaconda/ui/tui/spokes/storage.py:127
#: pyanaconda/ui/gui/spokes/storage.py:446
msgid "Error checking storage configuration"
msgstr ""
"Błąd podczas sprawdzania konfiguracji urządzeń do przechowywania danych"

#: pyanaconda/ui/tui/spokes/storage.py:130
#: pyanaconda/ui/gui/spokes/storage.py:448
msgid "Automatic partitioning selected"
msgstr "Wybrano automatyczne partycjonowanie"

#: pyanaconda/ui/tui/spokes/storage.py:132
#: pyanaconda/ui/gui/spokes/storage.py:450
msgid "Custom partitioning selected"
msgstr "Wybrano partycjonowanie użytkownika"

#: pyanaconda/ui/tui/spokes/storage.py:168
#: pyanaconda/ui/gui/spokes/storage.py:582
msgid ""
"No disks detected.  Please shut down the computer, connect at least one "
"disk, and restart to complete installation."
msgstr ""
"Nie wykryto dysku. Proszę wyłączyć komputer, podłączyć co najmniej jeden "
"dysk i uruchomić ponownie, aby ukończyć instalację."

#: pyanaconda/ui/tui/spokes/storage.py:170
#: pyanaconda/ui/gui/spokes/storage.py:584
msgid "No disks selected; please select at least one disk to install to."
msgstr "Nie wybrano dysku. Proszę wybrać co najmniej jeden dysk do instalacji."

#: pyanaconda/ui/tui/spokes/storage.py:185
#: pyanaconda/ui/gui/spokes/source.py:671
#: pyanaconda/ui/gui/spokes/storage.py:514
msgid "Probing storage..."
msgstr "Wykrywanie urządzeń do przechowywania danych..."

#: pyanaconda/ui/tui/spokes/storage.py:263
msgid "Generating updated storage configuration"
msgstr ""
"Tworzenie zaktualizowanej konfiguracji urządzeń do przechowywania danych"

#: pyanaconda/ui/tui/spokes/storage.py:268
#, python-format
msgid "storage configuration failed: %s"
msgstr "konfiguracja urządzeń do przechowywania danych się nie powiodła: %s"

#: pyanaconda/ui/tui/spokes/storage.py:277
#: pyanaconda/ui/gui/spokes/storage.py:307
msgid "Checking storage configuration..."
msgstr "Sprawdzanie konfiguracji urządzeń do przechowywania danych..."

#: pyanaconda/ui/tui/spokes/storage.py:311
msgid "Autopartitioning Options"
msgstr "Opcje automatycznego partycjonowania"

#: pyanaconda/ui/tui/spokes/storage.py:342
msgid ""
"Installation requires partitioning of your hard drive.  Select what space to "
"use for the install target."
msgstr ""
"Instalacja wymaga partycjonowania dysku twardego. Proszę wybrać, jakiego "
"miejsca użyć do instalacji."

#: pyanaconda/ui/tui/spokes/time.py:30
msgid "Timezone settings"
msgstr "Ustawienia strefy czasowej"

#: pyanaconda/ui/tui/spokes/time.py:53 pyanaconda/ui/tui/spokes/time.py:55
#: pyanaconda/ui/gui/spokes/datetime_spoke.py:379
#: pyanaconda/ui/gui/spokes/datetime_spoke.py:384
#, python-format
msgid "%s timezone"
msgstr "Strefa czasowa %s"

#: pyanaconda/ui/tui/spokes/time.py:57
msgid "Timezone is not set."
msgstr "Strefa czasowa nie jest ustawiona."

#: pyanaconda/ui/tui/spokes/time.py:65
#, python-format
msgid "Available timezones in region %s"
msgstr "Dostępne strefy czasowe w regionie %s"

#: pyanaconda/ui/tui/spokes/time.py:68
msgid "Available regions"
msgstr "Dostępne regiony"

#: pyanaconda/ui/tui/spokes/time.py:131
msgid ""
"Please select the timezone.\n"
"Use numbers or type names directly [b to region list, q to quit]: "
msgstr ""
"Proszę wybrać strefę czasową.\n"
"Należy użyć liczb lub podać nazwy bezpośrednio [b wyświetli listę regionów, "
"q zakończy]: "

#: pyanaconda/ui/tui/spokes/__init__.py:44
msgid "Default spoke title"
msgstr "Domyślny tytuł ekranu"

#: pyanaconda/ui/tui/spokes/__init__.py:54
msgid "testing status..."
msgstr "testowanie stanu..."

#: pyanaconda/ui/tui/__init__.py:58
msgid "Press enter to exit."
msgstr "Naciśnięcie klawisza Enter zakończy."

#: pyanaconda/ui/tui/__init__.py:67
msgid "Question"
msgstr "Pytanie"

#: pyanaconda/ui/tui/__init__.py:90
msgid "Please respond 'yes' or 'no': "
msgstr "Proszę odpowiedzieć \"tak\" lub \"nie\": "

#: pyanaconda/ui/tui/__init__.py:93
msgid "yes"
msgstr "tak"

#: pyanaconda/ui/tui/__init__.py:98
msgid "no"
msgstr "nie"

#: pyanaconda/ui/gui/__init__.py:144
msgid "_Exit Installer"
msgstr "Za_kończ instalator"

#: pyanaconda/ui/gui/__init__.py:153 pyanaconda/ui/gui/spokes/software.py:333
#: pyanaconda/ui/gui/spokes/storage.py:770 pyanaconda/ui/gui/main.glade:36
#: pyanaconda/ui/gui/hubs/summary.glade:77 widgets/src/StandaloneWindow.c:52
msgid "_Quit"
msgstr "Za_kończ"

#: pyanaconda/ui/gui/__init__.py:169
msgid "_No"
msgstr "_Nie"

#: pyanaconda/ui/gui/__init__.py:169
msgid "_Yes"
msgstr "_Tak"

#: pyanaconda/ui/gui/categories/__init__.py:44
msgid "DEFAULT TITLE"
msgstr "DOMYŚLNY TYTUŁ"

#: pyanaconda/ui/gui/categories/localization.py:31
msgid "LOCALIZATION"
msgstr "LOKALIZACJA"

#: pyanaconda/ui/gui/categories/software.py:31
msgid "SOFTWARE"
msgstr "OPROGRAMOWANIE"

#: pyanaconda/ui/gui/categories/storage.py:32
msgid "STORAGE"
msgstr "URZĄDZENIA DO PRZECHOWYWANIA DANYCH"

#: pyanaconda/ui/gui/categories/user_settings.py:32
msgid "USER SETTINGS"
msgstr "USTAWIENIA UŻYTKOWNIKA"

#: pyanaconda/ui/gui/hubs/__init__.py:226
msgid ""
"When all items marked with this icon are complete, installation will "
"automatically continue."
msgstr ""
"Kiedy wszystkie elementy oznaczone tą ikoną zostaną wypełnione, instalacja "
"zostanie automatycznie kontynuowana."

#: pyanaconda/ui/gui/hubs/__init__.py:228
msgid ""
"Please complete items marked with this icon before continuing to the next "
"step."
msgstr ""
"Proszę wypełnić elementy oznaczone tą ikoną przed kontynuowaniem do "
"następnego kroku."

#: pyanaconda/ui/gui/hubs/progress.py:254
msgid "Complete!"
msgstr "Ukończono."

#: pyanaconda/ui/gui/spokes/custom.py:99
#, python-format
msgid "New %s %s Installation"
msgstr "Nowa instalacja systemu %s %s"

#: pyanaconda/ui/gui/spokes/custom.py:100
msgid "Create a new volume group ..."
msgstr "Utwórz nową grupę woluminów..."

#: pyanaconda/ui/gui/spokes/custom.py:101
msgid ""
"Storage configuration reset due to unrecoverable error. Click for details."
msgstr ""
"Przywrócono konfigurację urządzeń do przechowywania danych z powodu "
"nieodwracalnego błędu. Kliknięcie wyświetli szczegóły."

#: pyanaconda/ui/gui/spokes/custom.py:103
msgid "Device reconfiguration failed. Click for details."
msgstr ""
"Ponowna konfiguracja urządzenia się nie powiodła. Kliknięcie wyświetli "
"szczegóły."

#: pyanaconda/ui/gui/spokes/custom.py:106
msgid "Please enter a valid mountpoint."
msgstr "Proszę podać prawidłowy punkt montowania."

#: pyanaconda/ui/gui/spokes/custom.py:107
msgid "That mount point is invalid. Try something else?"
msgstr "Punkt montowania jest nieprawidłowy. Proszę spróbować użyć innego."

#: pyanaconda/ui/gui/spokes/custom.py:108
msgid "That mount point is already in use. Try something else?"
msgstr "Punkt montowania jest już używany. Proszę spróbować użyć innego."

#: pyanaconda/ui/gui/spokes/custom.py:120
msgid "LVM"
msgstr "LVM"

#: pyanaconda/ui/gui/spokes/custom.py:121
msgid "RAID"
msgstr "RAID"

#: pyanaconda/ui/gui/spokes/custom.py:122
msgid "Standard Partition"
msgstr "Standardowa partycja"

#: pyanaconda/ui/gui/spokes/custom.py:123
msgid "BTRFS"
msgstr "BTRFS"

#: pyanaconda/ui/gui/spokes/custom.py:124
#: pyanaconda/ui/gui/spokes/custom.glade:2088
#: pyanaconda/ui/gui/spokes/custom.glade:2289
#: pyanaconda/ui/gui/spokes/lib/resize.glade:140
msgid "Disk"
msgstr "Dysk"

#: pyanaconda/ui/gui/spokes/custom.py:439
#: pyanaconda/ui/gui/spokes/custom.glade:389
msgid "MANUAL PARTITIONING"
msgstr "RĘCZNE PARTYCJONOWANIE"

#: pyanaconda/ui/gui/spokes/custom.py:636
#, python-format
msgid "%d storage device selected"
msgid_plural "%d storage devices selected"
msgstr[0] "Wykryto %d urządzenie do przechowywania danych"
msgstr[1] "Wykryto %d urządzenia do przechowywania danych"
msgstr[2] "Wykryto %d urządzeń do przechowywania danych"

#: pyanaconda/ui/gui/spokes/custom.py:789
#: pyanaconda/ui/gui/spokes/network.py:585
#: pyanaconda/ui/gui/spokes/network.py:1001
msgid "Unknown"
msgstr "Nieznane"

#: pyanaconda/ui/gui/spokes/custom.py:818
msgid ""
"The 'swap' area on your computer is used by the operating\n"
"system when running low on memory."
msgstr ""
"Obszar wymiany (\"swap\") komputera jest używany przez\n"
"system operacyjny wtedy, gdy zabraknie pamięci."

#: pyanaconda/ui/gui/spokes/custom.py:821
msgid ""
"The 'boot' area on your computer is where files needed\n"
"to start the operating system are stored."
msgstr ""
"Obszar startowy (\"boot\") komputera jest miejscem, gdzie\n"
"przechowywane są pliki wymagane do uruchomienia systemu."

#: pyanaconda/ui/gui/spokes/custom.py:824
msgid ""
"The 'root' area on your computer is where core system\n"
"files and applications are stored."
msgstr ""
"Obszar \"root\" komputera jest miejscem, gdzie\n"
"przechowywane są główne pliki systemu i aplikacje."

#: pyanaconda/ui/gui/spokes/custom.py:827
msgid ""
"The 'home' area on your computer is where all your personal\n"
"data is stored."
msgstr ""
"Obszar domowy (\"home\") komputera jest miejscem, gdzie\n"
"przechowywane są wszystkie osobiste dane użytkownika."

#: pyanaconda/ui/gui/spokes/custom.py:830
msgid ""
"The BIOS boot partition is required to enable booting\n"
"from GPT-partitioned disks on BIOS hardware."
msgstr ""
"Partycja startowa BIOS jest wymagana, aby umożliwić uruchamianie \n"
"z dysków partycjonowanych jako GPT na sprzęcie typu BIOS."

#: pyanaconda/ui/gui/spokes/custom.py:833
msgid ""
"The PReP boot partition is required as part of the\n"
"bootloader configuration on some PPC platforms."
msgstr ""
"Partycja startowa PReP jest wymagana jako część konfiguracji\n"
"programu startowego na niektórych platformach PPC."

#: pyanaconda/ui/gui/spokes/custom.py:995
#, python-format
msgid "/boot/efi must be on a device of type %s"
msgstr "/boot/efi musi być na urządzeniu typu %s"

#: pyanaconda/ui/gui/spokes/custom.py:999
#, python-format
msgid "%s must be on a device of type %s"
msgstr "%s musi być na urządzeniu typu %s"

#: pyanaconda/ui/gui/spokes/custom.py:1002
#: pyanaconda/ui/gui/spokes/custom.py:1004
#, python-format
msgid "%s cannot be encrypted"
msgstr "%s nie może być szyfrowane"

#: pyanaconda/ui/gui/spokes/custom.py:1008
#, python-format
msgid "Devices of type %s require a valid RAID level selection."
msgstr "Urządzenia typu %s wymagają prawidłowego wyboru poziomu RAID."

#: pyanaconda/ui/gui/spokes/custom.py:1014
msgid ""
"The RAID level you have selected requires more disks than you currently have "
"selected."
msgstr "Wybrany poziom RAID wymaga więcej dysków, niż obecnie wybrano."

#: pyanaconda/ui/gui/spokes/custom.py:1173
msgid "Device resize request failed. Click for details."
msgstr ""
"Żądanie zmiany rozmiaru urządzenia się nie powiodło. Kliknięcie wyświetli "
"szczegóły."

#: pyanaconda/ui/gui/spokes/custom.py:1227
#, python-format
msgid "Specified name %s already in use."
msgstr "Podana nazwa %s jest już używana."

#: pyanaconda/ui/gui/spokes/custom.py:1294
msgid "Device reformat request failed. Click for details."
msgstr ""
"Żądanie ponownego sformatowania urządzenia się nie powiodło. Kliknięcie "
"wyświetli szczegóły."

#: pyanaconda/ui/gui/spokes/custom.py:1558
msgid "This file system does not support labels."
msgstr "Ten system plików nie obsługuje etykiet."

#: pyanaconda/ui/gui/spokes/custom.py:1576
msgid "This file system may not be resized."
msgstr "Rozmiar tego systemu plików nie może być zmieniany."

#: pyanaconda/ui/gui/spokes/custom.py:1848
#, python-format
msgid "Added new %s to existing container %s."
msgstr "Dodano nową %s do istniejącego kontenera %s."

#: pyanaconda/ui/gui/spokes/custom.py:1856
msgid "Failed to add new device. Click for details."
msgstr ""
"Dodanie nowego urządzenia się nie powiodło. Kliknięcie wyświetli szczegóły."

#: pyanaconda/ui/gui/spokes/custom.py:1862
msgid "Invalid partition size set. Use a valid integer."
msgstr ""
"Ustawiono nieprawidłowy rozmiar partycji. Należy użyć prawidłowej liczby "
"całkowitej."

#: pyanaconda/ui/gui/spokes/custom.py:1882
msgid "Device removal request failed. Click for details."
msgstr ""
"Żądanie usunięcia urządzenia się nie powiodło. Kliknięcie wyświetli "
"szczegóły."

#: pyanaconda/ui/gui/spokes/custom.py:2077
#, python-format
msgid "Volume Group name %s is already in use. Not saving changes."
msgstr "Nawa grupy woluminów %s jest już używana. Zmiany nie zostaną zapisane."

#: pyanaconda/ui/gui/spokes/custom.py:2168
#, python-format
msgid ""
"This Software RAID array is missing %d of %d member partitions. You can "
"remove it or select a different device."
msgstr ""
"W tej macierzy programowego RAID brakuje %d z %d partycji. Można ją usunąć "
"lub wybrać inną."

#: pyanaconda/ui/gui/spokes/custom.py:2174
#, python-format
msgid ""
"This LVM Volume Group is missing %d of %d physical volumes. You can remove "
"it or select a different device."
msgstr ""
"W tej grupie woluminów LVM brakuje %d z %d woluminów fizycznych. Można ją "
"usunąć lub wybrać inną."

#: pyanaconda/ui/gui/spokes/custom.py:2247
msgid "No disks selected."
msgstr "Nie wybrano dysku."

#: pyanaconda/ui/gui/spokes/custom.py:2253
msgid "Not enough free space on selected disks."
msgstr "Brak wystarczającej ilości wolnego miejsca na zaznaczonych dyskach."

#: pyanaconda/ui/gui/spokes/custom.py:2259
msgid "Automatic partitioning failed. Click for details."
msgstr ""
"Automatyczne partycjonowanie się nie powiodło. Kliknięcie wyświetli "
"szczegóły."

#: pyanaconda/ui/gui/spokes/custom.py:2499
msgid "Failed to unlock encrypted block device. Click for details"
msgstr ""
"Odblokowanie zaszyfrowanego urządzenia blokowego się nie powiodło. "
"Kliknięcie wyświetli szczegóły."

#: pyanaconda/ui/gui/spokes/custom.py:2522
#, python-format
msgid ""
"You have chosen to manually set up the filesystems for your new "
"%(productName)s installation. Before you begin, you might want to take a "
"minute to learn the lay of the land. Quite a bit has changed.\n"
"\n"
"The most important change is that creation of new filesystems has been "
"streamlined. You no longer have to build complex devices like LVM logical "
"volumes in stages (physical volume, then volume group, then logical volume) "
"-- now you just create a logical volume and we'll handle the legwork of "
"setting up the physical volumes and volume group to contain it. We'll also "
"handle adjusting the volume group as you add, remove, and resize logical "
"volumes so you don't have to worry about the mundane details.\n"
"\n"
"\n"
"Screen Layout\n"
"\n"
"The left-hand side of the screen shows the OS installations we were able to "
"find on this computer. The new %(productName)s installation is at the top of "
"the list. You can click on the names of the installations to see what "
"filesystems they contain.\n"
"\n"
"Below the various installations and mountpoints on the left-hand side there "
"are buttons to add a new filesystem, remove the selected filesystem, or "
"configure the selected filesystem.\n"
"\n"
"The right-hand side of the screen is where you can customize the currently-"
"selected mountpoint.\n"
"\n"
"On the bottom-left you will see a summary of the disks you have chosen to "
"use for the installation. You can click on the blue text to see more "
"detailed information about your selected disks.\n"
"\n"
"\n"
"How to create a new filesystem on a new device\n"
"\n"
"1. Click on the + button.\n"
"2. Enter the mountpoint and size. (Hint: Hover the mouse pointer over either "
"of the text entry areas for help.)\n"
"3. Select the new mountpoint under \"New %(productName)s Installation\" on "
"the left-hand side of the screen and customize it to suit your needs.\n"
"\n"
"\n"
"How to reformat a device/filesystem that already exists on your disk\n"
"\n"
"1. Select the filesystem from the left-hand side of the screen.\n"
"2. Click on the \"Customize\" expander in the mountpoint customization area "
"on the right-hand side of the screen.\n"
"3. Activate the \"Reformat\" checkbutton, select a filesystem type and, if "
"applicable, enter a mountpoint above in the \"Mountpoint\" text entry area.\n"
"4. Click on \"Apply changes\"\n"
"\n"
"\n"
"How to set a mountpoint for a filesystem that already exists on your disk\n"
"\n"
"1. Select the filesystem from the left-hand side of the screen.\n"
"2. Enter a mountpoint in the \"Mountpoint\" text entry area in the "
"mountpoint customization area.\n"
"3. Click on \"Apply changes\"\n"
"\n"
"\n"
"How to remove a filesystem that already exists on your disk\n"
"\n"
"1. Select the filesystem you wish to remove on the left-hand side of the "
"screen.\n"
"2. Click the - button.\n"
"\n"
"Hint: Removing a device that already exists on your disk from the \"New "
"%(productName)s Installation\" does not remove it from the disk. It only "
"resets that device to its original state. To remove a device that already "
"exists on your disk, you must select it from under any of the other detected "
"installations (or \"Unknown\") and hit the - button.\n"
"\n"
"\n"
"Tips and hints\n"
"\n"
"You can enter sizes for new filesystems that are greater than the total "
"available free space. The installer will come as close as possible to the "
"size you request.\n"
"\n"
"By default, new devices use any/all of your selected disks.\n"
"\n"
"You can change which disks a new device may be allocated from by clicking "
"the configure button (the one with a tools graphic) while that device is "
"selected.\n"
"\n"
"When adding a new mountpoint by clicking the + button, leave the size entry "
"blank to make the new device use all available free space.\n"
"\n"
"When you remove the last device from a container device like an LVM volume "
"group, we will automatically remove that container device to make room for "
"new devices.\n"
"\n"
"When the last partition is removed from a disk, that disk may be "
"reinitialized with a new partition table if we think there is a more "
"appropriate type for that disk.\n"
msgstr ""

#: pyanaconda/ui/gui/spokes/datetime_spoke.py:56
msgid "Note: pool servers may not be available all the time"
msgstr "Uwaga: serwery puli mogą nie być dostępne przez cały czas"

#: pyanaconda/ui/gui/spokes/datetime_spoke.py:296
#: pyanaconda/ui/gui/spokes/datetime_spoke.glade:31
msgid "DATE & TIME"
msgstr "DATA I CZAS"

#: pyanaconda/ui/gui/spokes/datetime_spoke.py:381
msgid "Invalid timezone"
msgstr "Nieprawidłowa strefa czasowa"

#: pyanaconda/ui/gui/spokes/datetime_spoke.py:813
msgid "You need to set up networking first if you want to use NTP"
msgstr "Należy najpierw ustawić sieć, jeśli ma być używane NTP"

#: pyanaconda/ui/gui/spokes/datetime_spoke.py:817
msgid "You have no working NTP server configured"
msgstr "Nie skonfigurowano działającego serwera NTP"

#: pyanaconda/ui/gui/spokes/keyboard.py:40
#, python-format
msgid "%s to switch layouts."
msgstr "%s przełącza układy."

#: pyanaconda/ui/gui/spokes/keyboard.py:261
msgid "KEYBOARD"
msgstr "KLAWIATURA"

#: pyanaconda/ui/gui/spokes/keyboard.py:318
msgid "Testing layouts configuration not available."
msgstr "Testowanie konfiguracji układów jest niedostępne."

#: pyanaconda/ui/gui/spokes/keyboard.py:372
msgid "Layout switching not configured."
msgstr "Przełączanie układów nie jest skonfigurowane."

#: pyanaconda/ui/gui/spokes/network.py:92
msgid "Status unknown (missing)"
msgstr "Nieznany stan (brak)"

#: pyanaconda/ui/gui/spokes/network.py:96
msgid "Status unknown"
msgstr "Nieznany stan"

#: pyanaconda/ui/gui/spokes/network.py:98
msgid "Unmanaged"
msgstr "Niezarządzane"

#: pyanaconda/ui/gui/spokes/network.py:101
msgid "Firmware missing"
msgstr "Brak oprogramowania sprzętowego"

#: pyanaconda/ui/gui/spokes/network.py:104
msgid "Cable unplugged"
msgstr "Kabel jest niepodłączony"

#: pyanaconda/ui/gui/spokes/network.py:106
msgid "Unavailable"
msgstr "Niedostępne"

#: pyanaconda/ui/gui/spokes/network.py:108
msgid "Disconnected"
msgstr "Rozłączono"

#: pyanaconda/ui/gui/spokes/network.py:113
msgid "Connecting"
msgstr "Łączenie"

#: pyanaconda/ui/gui/spokes/network.py:115
msgid "Authentication required"
msgstr "Wymagane jest uwierzytelnienie"

#: pyanaconda/ui/gui/spokes/network.py:117
msgid "Connected"
msgstr "Połączono"

#: pyanaconda/ui/gui/spokes/network.py:119
msgid "Disconnecting"
msgstr "Rozłączanie"

#: pyanaconda/ui/gui/spokes/network.py:121
msgid "Connection failed"
msgstr "Połączenie się nie powiodło"

#: pyanaconda/ui/gui/spokes/network.py:574
msgid "unplugged"
msgstr "niepodłączone"

#: pyanaconda/ui/gui/spokes/network.py:587
msgid "Ethernet"
msgstr "Ethernet"

#: pyanaconda/ui/gui/spokes/network.py:589
msgid "Wireless"
msgstr "Bezprzewodowe"

#: pyanaconda/ui/gui/spokes/network.py:683
#: pyanaconda/ui/gui/spokes/network.glade:349
#: pyanaconda/ui/gui/spokes/network.glade:702
msgid "IPv4 Address"
msgstr "Adres IPv4"

#: pyanaconda/ui/gui/spokes/network.py:684
#: pyanaconda/ui/gui/spokes/network.glade:363
#: pyanaconda/ui/gui/spokes/network.glade:716
#: pyanaconda/ui/gui/spokes/network.glade:1335
msgid "IPv6 Address"
msgstr "Adres IPv6"

#: pyanaconda/ui/gui/spokes/network.py:686
#: pyanaconda/ui/gui/spokes/network.py:688
#: pyanaconda/ui/gui/spokes/network.glade:1321
msgid "IP Address"
msgstr "Adres IP"

#: pyanaconda/ui/gui/spokes/network.py:753
#, python-format
msgid "%d Mb/s"
msgstr "%d Mb/s"

#: pyanaconda/ui/gui/spokes/network.py:930
msgid "WEP"
msgstr "WEP"

#: pyanaconda/ui/gui/spokes/network.py:933
msgid "WPA"
msgstr "WPA"

#: pyanaconda/ui/gui/spokes/network.py:936
msgid "WPA2"
msgstr "WPA2"

#: pyanaconda/ui/gui/spokes/network.py:940
msgid "Enterprise"
msgstr "Enterprise"

#: pyanaconda/ui/gui/spokes/network.py:945 widgets/src/SpokeSelector.c:54
msgid "None"
msgstr "Brak"

#: pyanaconda/ui/gui/spokes/network.py:967
#: pyanaconda/ui/gui/spokes/network.glade:37
#: pyanaconda/ui/gui/spokes/network.glade:109
msgid "NETWORK CONFIGURATION"
msgstr "KONFIGURACJA SIECI"

#: pyanaconda/ui/gui/spokes/network.py:1005
msgid "Connecting..."
msgstr "Łączenie..."

#: pyanaconda/ui/gui/spokes/network.py:1007
msgid "Disconnecting..."
msgstr "Rozłączanie..."

#: pyanaconda/ui/gui/spokes/network.py:1014
#, python-format
msgid "Wireless connected to %s"
msgstr "Bezprzewodowe połączono z %s"

#: pyanaconda/ui/gui/spokes/network.py:1017
#, python-format
msgid "Wired (%s) connected"
msgstr "Połączono przewodowe (%s)"

#: pyanaconda/ui/gui/spokes/network.py:1027
#, python-format
msgid "Connected: %s"
msgstr "Połączono: %s"

#: pyanaconda/ui/gui/spokes/network.py:1029
msgid "Not connected"
msgstr "Nie połączono"

#: pyanaconda/ui/gui/spokes/network.py:1032
#: pyanaconda/ui/gui/spokes/network.glade:2085
msgid "No network devices available"
msgstr "Brak dostępnych urządzeń sieciowych"

#: pyanaconda/ui/gui/spokes/network.py:1072
#: pyanaconda/ui/gui/spokes/network.py:1142
#, python-format
msgid "Hostname is not valid: %s"
msgstr "Nazwa komputera jest nieprawidłowa: %s"

#: pyanaconda/ui/gui/spokes/password.py:48
#: pyanaconda/ui/gui/spokes/password.glade:9
msgid "ROOT PASSWORD"
msgstr "HASŁO ROOTA"

#: pyanaconda/ui/gui/spokes/password.py:77
msgid "Error setting root password"
msgstr "Błąd podczas ustawiania hasła roota"

#: pyanaconda/ui/gui/spokes/password.py:79
msgid "Root password is set"
msgstr "Hasło roota jest ustawione"

#: pyanaconda/ui/gui/spokes/password.py:81
msgid "Root account is disabled"
msgstr "Konto roota jest wyłączone"

#: pyanaconda/ui/gui/spokes/password.py:83
msgid "Root password is not set"
msgstr "Hasło roota nie jest ustawione"

#: pyanaconda/ui/gui/spokes/password.py:103
msgid "You must provide and confirm a password."
msgstr "Należy podać i potwierdzić hasło."

#: pyanaconda/ui/gui/spokes/password.py:113
#, python-format
msgid "You have provided a weak password: %s.  Press Done again to use anyway."
msgstr ""
"Podano słabe hasło: %s. Proszę nacisnąć przycisk Gotowe ponownie, aby użyć "
"je mimo to."

#: pyanaconda/ui/gui/spokes/software.py:53
#: pyanaconda/ui/gui/spokes/software.glade:31
msgid "SOFTWARE SELECTION"
msgstr "WYBÓR OPROGRAMOWANIA"

#: pyanaconda/ui/gui/spokes/software.py:107
#: pyanaconda/ui/gui/spokes/source.py:612
msgid "Checking software dependencies..."
msgstr "Sprawdzanie zależności oprogramowania..."

#: pyanaconda/ui/gui/spokes/software.py:113
msgid "Error checking software dependencies"
msgstr "Błąd podczas sprawdzania zależności oprogramowania"

#: pyanaconda/ui/gui/spokes/software.py:150
msgid "Error checking software selection"
msgstr "Błąd podczas sprawdzania wyboru oprogramowania"

#: pyanaconda/ui/gui/spokes/software.py:153
msgid "Installation source not set up"
msgstr "Nie ustawiono źródła instalacji"

#: pyanaconda/ui/gui/spokes/software.py:156
msgid "Source changed - please verify"
msgstr "Zmieniono źródło - proszę sprawdzić"

#: pyanaconda/ui/gui/spokes/software.py:164
msgid "Custom software selected"
msgstr "Wybrano oprogramowanie użytkownika"

#: pyanaconda/ui/gui/spokes/software.py:166
#: pyanaconda/ui/gui/spokes/source.py:597
#: pyanaconda/ui/gui/spokes/source.py:630
msgid "Nothing selected"
msgstr "Nic nie wybrano"

#: pyanaconda/ui/gui/spokes/software.py:175
#: pyanaconda/ui/gui/spokes/source.py:50
msgid "Downloading package metadata..."
msgstr "Pobieranie metadanych pakietów..."

#: pyanaconda/ui/gui/spokes/software.py:181
msgid "Downloading group metadata..."
msgstr "Pobieranie metadanych grup..."

#: pyanaconda/ui/gui/spokes/software.py:215
#: pyanaconda/ui/gui/spokes/source.py:589
msgid "No installation source available"
msgstr "Brak dostępnych źródeł instalacji"

#: pyanaconda/ui/gui/spokes/software.py:265
msgid "Error checking software dependencies.  Click for details."
msgstr ""
"Błąd podczas sprawdzania zależności oprogramowania. Kliknięcie wyświetli "
"więcej informacji."

#: pyanaconda/ui/gui/spokes/software.py:329
msgid ""
"The following software marked for installation has errors.  This is likely "
"caused by an error with\n"
"your installation source.  You can attempt to remove these packages from "
"your installation.\n"
"change your installation source, or quit the installer."
msgstr ""
"Poniższe oprogramowanie zaznaczone do instalacji posiada błędy. Jest to "
"prawdopodobnie spowodowane\n"
"przez błąd w źródle instalacji. Można spróbować usunąć te pakiety z "
"instalacji, zmienić źródło\n"
"instalacji lub zakończyć instalator."

#: pyanaconda/ui/gui/spokes/software.py:333
#: pyanaconda/ui/gui/spokes/storage.glade:613
#: pyanaconda/ui/gui/spokes/source.glade:592
#: pyanaconda/ui/gui/spokes/custom.glade:91
#: pyanaconda/ui/gui/spokes/custom.glade:341
#: pyanaconda/ui/gui/spokes/custom.glade:1956
#: pyanaconda/ui/gui/spokes/custom.glade:2194 pyanaconda/ui/gui/main.glade:20
msgid "_Cancel"
msgstr "_Anuluj"

#: pyanaconda/ui/gui/spokes/software.py:334
msgid "_Remove Packages"
msgstr "_Usuń pakiety"

#: pyanaconda/ui/gui/spokes/software.py:335
msgid "_Modify Software Source"
msgstr "_Modyfikuj źródło oprogramowania"

#: pyanaconda/ui/gui/spokes/source.py:49
msgid "Setting up installation source..."
msgstr "Ustawianie źródła instalacji..."

#: pyanaconda/ui/gui/spokes/source.py:51
msgid "Error downloading package metadata..."
msgstr "Błąd podczas pobierania metadanych pakietów..."

#: pyanaconda/ui/gui/spokes/source.py:144
msgid "This media is good to install from."
msgstr "Z tego nośnika można instalować."

#: pyanaconda/ui/gui/spokes/source.py:146
msgid "This media is not good to install from."
msgstr "Z tego nośnika nie można instalować."

#: pyanaconda/ui/gui/spokes/source.py:347
msgid "Getting info about requested repository"
msgstr "Pobieranie informacji o żądanym repozytorium"

#: pyanaconda/ui/gui/spokes/source.py:370
#: pyanaconda/ui/gui/spokes/source.glade:257
msgid "enter URL of your desired repository"
msgstr "proszę podać adres URL żądanego repozytorium"

#: pyanaconda/ui/gui/spokes/source.py:428
#: pyanaconda/ui/gui/spokes/source.glade:827
msgid "INSTALLATION SOURCE"
msgstr "ŹRÓDŁO INSTALACJI"

#: pyanaconda/ui/gui/spokes/source.py:561
msgid "Failed to set up install source"
msgstr "Ustawienie źródła instalacji się nie powiodło"

#: pyanaconda/ui/gui/spokes/source.py:563
#: pyanaconda/ui/gui/spokes/source.py:577
msgid "Failed to set up install source, check the repo url"
msgstr ""
"Ustawienie źródła instalacji się nie powiodło. Proszę sprawdzić adres URL "
"repozytorium"

#: pyanaconda/ui/gui/spokes/source.py:565
msgid "Failed to set up install source, check the repo url and proxy settings"
msgstr ""
"Ustawienie źródła instalacji się nie powiodło. Proszę sprawdzić adres URL "
"repozytorium i ustawienia pośrednika"

#: pyanaconda/ui/gui/spokes/source.py:614
msgid "Not ready"
msgstr "Niegotowe"

#: pyanaconda/ui/gui/spokes/source.py:616
#: pyanaconda/ui/gui/spokes/source.py:625
msgid "Error setting up software source"
msgstr "Błąd podczas ustawiania źródła oprogramowania"

#: pyanaconda/ui/gui/spokes/source.py:620
#, python-format
msgid "NFS server %s"
msgstr "Serwer NFS %s"

#: pyanaconda/ui/gui/spokes/source.py:622
msgid "CD/DVD drive"
msgstr "Napęd CD/DVD"

#: pyanaconda/ui/gui/spokes/source.py:628
msgid "Closest mirror"
msgstr "Najbliższy serwer źródlany"

#: pyanaconda/ui/gui/spokes/storage.py:151
#, python-format
msgid ""
"You have plenty of space to install <b>%s</b>, so we can automatically\n"
"configure the rest of the installation for you.\n"
"\n"
"You're all set!"
msgstr ""
"Jest wystarczająca ilość miejsca, aby zainstalować <b>%s</b>, więc można\n"
"automatycznie skonfigurować resztę instalacji.\n"
"\n"
"Wszystko gotowe!"

#: pyanaconda/ui/gui/spokes/storage.py:170
#, python-format
msgid ""
"Your current <b>%s</b> software selection requires <b>%s</b> of available "
"space."
msgstr ""
"Obecny wybór oprogramowania <b>%s</b> wymaga <b>%s</b> dostępnego miejsca."

#: pyanaconda/ui/gui/spokes/storage.py:243
#, python-format
msgid ""
"%s\n"
"The disks you've selected have the following amounts of free space:"
msgstr ""
"%s\n"
"Wybrane dyski posiadają następujące ilości wolnego miejsca:"

#: pyanaconda/ui/gui/spokes/storage.py:252
#, python-format
msgid ""
"<b>You don't have enough space available to install %s</b>, but we can help "
"you\n"
"reclaim space by shrinking or removing existing partitions."
msgstr ""
"<b>Brak wystarczającego miejsca, aby zainstalować %s</b>, ale można\n"
"odzyskać je przez zmniejszenie lub usunięcie istniejących partycji."

#: pyanaconda/ui/gui/spokes/storage.py:274
#, python-format
msgid ""
"%s\n"
"You don't have enough space available to install <b>%s</b>, even if you used "
"all of the free space\n"
"available on the selected disks."
msgstr ""
"%s\n"
"Brak wystarczającego miejsca, aby zainstalować <b>%s</b>, nawet jeśli użyte\n"
"zostanie całe wolne miejsce dostępne na wybranych dyskach."

#: pyanaconda/ui/gui/spokes/storage.py:285
#, python-format
msgid ""
"<b>You don't have enough space available to install %s</b>, even if you used "
"all of the free space\n"
"available on the selected disks.  You could add more disks for additional "
"space,\n"
"modify your software selection to install a smaller version of <b>%s</b>, or "
"quit the installer."
msgstr ""
"<b>Brak wystarczającego miejsca, aby zainstalować %s</b>, nawet jeśli użyte\n"
"zostanie całe wolne miejsce dostępne na wybranych dyskach. Można dodać\n"
"więcej dysków, aby uzyskać dodatkowe miejsce, zmodyfikować wybór\n"
"oprogramowania, aby zainstalować mniejszą wersję systemu <b>%s</b> lub\n"
"zakończyć instalator."

#: pyanaconda/ui/gui/spokes/storage.py:325
#: pyanaconda/ui/gui/spokes/storage.glade:844
msgid "INSTALLATION DESTINATION"
msgstr "CEL INSTALACJI"

#: pyanaconda/ui/gui/spokes/storage.py:397
msgid "Saving storage configuration..."
msgstr "Zapisywanie konfiguracji urządzeń do przechowywania danych..."

#: pyanaconda/ui/gui/spokes/storage.py:404
msgid "Failed to save storage configuration..."
msgstr ""
"Zapisanie konfiguracji urządzeń do przechowywania danych się nie powiodło..."

#: pyanaconda/ui/gui/spokes/storage.py:496
msgid "Error checking storage configuration.  Click for details."
msgstr ""
"Błąd podczas sprawdzania konfiguracji urządzeń do przechowywania danych. "
"Kliknięcie wyświetli więcej informacji."

#: pyanaconda/ui/gui/spokes/storage.py:575
#, python-format
msgid "%d disk selected; %s capacity; %s free"
msgid_plural "%d disks selected; %s capacity; %s free"
msgstr[0] "Wybrano %d dysk; pojemność %s; %s wolnego miejsca"
msgstr[1] "Wybrano %d dyski; pojemność %s; %s wolnego miejsca"
msgstr[2] "Wybrano %d dysków; pojemność %s; %s wolnego miejsca"

#: pyanaconda/ui/gui/spokes/storage.py:627
msgid ""
"You have chosen to skip bootloader installation.  Your system may not be "
"bootable."
msgstr ""
"Wybrano pominięcie instalacji programu startowego. Być może nie będzie można "
"uruchomić systemu."

#: pyanaconda/ui/gui/spokes/storage.py:767
msgid ""
"The following errors were encountered when checking your storage "
"configuration.  You can modify your storage layout\n"
"or quit the installer."
msgstr ""
"Poniższe błędy wystąpiły podczas sprawdzania konfiguracji urządzeń do "
"przechowywania danych. Można zmodyfikować układ\n"
"partycjonowania lub zakończyć instalator."

#: pyanaconda/ui/gui/spokes/storage.py:770
msgid "_Modify Storage Layout"
msgstr "Z_modyfikuj układ urządzeń do przechowywania danych"

#: pyanaconda/ui/gui/spokes/welcome.py:310
msgid "LANGUAGE"
msgstr "JĘZYK"

#: pyanaconda/ui/gui/spokes/lib/accordion.py:130
msgid "DATA"
msgstr "DANE"

#: pyanaconda/ui/gui/spokes/lib/accordion.py:135
msgid "SYSTEM"
msgstr "SYSTEM"

#: pyanaconda/ui/gui/spokes/lib/accordion.py:235
#, python-format
msgid "You haven't created any mount points for your %s %s installation yet:"
msgstr ""
"Nie utworzono jeszcze żadnych punktów montowania dla instalacji systemu %s "
"%s:"

#: pyanaconda/ui/gui/spokes/lib/accordion.py:240
msgid "Click here to create them automatically."
msgstr "Kliknięcie tutaj utworzy je automatycznie."

#: pyanaconda/ui/gui/spokes/lib/accordion.py:250
msgid "Or, create new mount points below with the '+' icon."
msgstr ""
"Można też poniżej utworzyć nowe punkty montowania za pomocą ikony \"+\"."

#: pyanaconda/ui/gui/spokes/lib/cart.py:138
#, python-format
msgid ""
"<b>%d disk; %s capacity; %s free space</b> (unpartitioned and in filesystems)"
msgid_plural ""
"<b>%d disks; %s capacity; %s free space</b> (unpartitioned and in "
"filesystems)"
msgstr[0] ""
"<b>%d dysk; pojemność %s; %s wolnego miejsca</b> (niepartycjonowane i na "
"systemach plików)"
msgstr[1] ""
"<b>%d dyski; pojemność %s; %s wolnego miejsca</b> (niepartycjonowane i na "
"systemach plików)"
msgstr[2] ""
"<b>%d dysków; pojemność %s; %s wolnego miejsca</b> (niepartycjonowane i na "
"systemach plików)"

#: pyanaconda/ui/gui/spokes/lib/cart.py:190
msgid "_Do not install bootloader"
msgstr "_Bez instalowania programu startowego"

#: pyanaconda/ui/gui/spokes/lib/cart.py:192
#: pyanaconda/ui/gui/spokes/lib/cart.glade:190
msgid "_Set as Boot Device"
msgstr "_Ustaw jako urządzenie startowe"

#: pyanaconda/ui/gui/spokes/lib/passphrase.py:38
#, python-format
msgid ""
"Warning: Your current keyboard layout is <b>%s</b>.If you change your "
"keyboard layout, you may not be able to decrypt your disks after install."
msgstr ""
"Ostrzeżenie: obecny układ klawiatury to <b>%s</b>. Jeśli zostanie on "
"zmieniony, to odszyfrowanie dysków po instalacji może nie być możliwe."

#: pyanaconda/ui/gui/spokes/lib/passphrase.py:42
#, python-format
msgid "You have provided a weak passphrase: %s"
msgstr "Podano słabe hasło: %s"

#: pyanaconda/ui/gui/spokes/lib/passphrase.py:43
msgid "Passphrases do not match."
msgstr "Hasła się nie zgadzają."

#: pyanaconda/ui/gui/spokes/lib/passphrase.py:116
#: pyanaconda/ui/gui/spokes/lib/passphrase.glade:165
msgid "Weak"
msgstr "Słabe"

#: pyanaconda/ui/gui/spokes/lib/passphrase.py:119
msgid "Fair"
msgstr "Dostateczne"

#: pyanaconda/ui/gui/spokes/lib/passphrase.py:122
msgid "Good"
msgstr "Dobre"

#: pyanaconda/ui/gui/spokes/lib/passphrase.py:125
msgid "Strong"
msgstr "Silne"

#: pyanaconda/ui/gui/spokes/lib/resize.py:46
#: pyanaconda/ui/gui/spokes/lib/resize.glade:216
msgid "Preserve"
msgstr "Zachowaj"

#: pyanaconda/ui/gui/spokes/lib/resize.py:47
msgid "Shrink"
msgstr "Zmniejsz"

#: pyanaconda/ui/gui/spokes/lib/resize.py:48
#: pyanaconda/ui/gui/spokes/lib/resize.glade:228
msgid "Delete"
msgstr "Usuń"

#: pyanaconda/ui/gui/spokes/lib/resize.py:134
msgid "Whole disks are not editable."
msgstr "Całych dysków nie można modyfikować."

#: pyanaconda/ui/gui/spokes/lib/resize.py:151
msgid "This device contains the installation source."
msgstr "To urządzenie zawiera źródło instalacji."

#: pyanaconda/ui/gui/spokes/lib/resize.py:158
#, python-format
msgid "%s of %s"
msgstr "%s z %s"

#: pyanaconda/ui/gui/spokes/lib/resize.py:175
msgid ""
"You don't have enough free space available for this installation.\n"
"\n"
"You can remove existing filesystems you no longer need to free up space for "
"this installation.  Removing a filesystem will permanently delete all of the "
"data it contains."
msgstr ""
"Brak wystarczającej ilości wolnego miejsca dla tej instalacji.\n"
"\n"
"Można usunąć już niepotrzebne istniejące systemy plików, aby zwolnić miejsce "
"dla tej instalacji. Usunięcie systemu plików trwale usunie wszystkie "
"znajdujące się na nim dane."

#: pyanaconda/ui/gui/spokes/lib/resize.py:191
#, python-format
msgid "<b>%s disk; %s reclaimable space</b> (in filesystems)"
msgid_plural "<b>%s disks; %s reclaimable space</b> (in filesystems)"
msgstr[0] ""
"<b>%s dysk; %s miejsca możliwego do odzyskania</b> (w systemach plików)"
msgstr[1] ""
"<b>%s dyski; %s miejsca możliwego do odzyskania</b> (w systemach plików)"
msgstr[2] ""
"<b>%s dysków; %s miejsca możliwego do odzyskania</b> (w systemach plików)"

#: pyanaconda/ui/gui/spokes/lib/resize.py:197
#, python-format
msgid "Total selected space to reclaim: <b>%s</b>"
msgstr "Razem wybranego miejsca do odzyskania: <b>%s</b>"

#: pyanaconda/ui/gui/spokes/datetime_spoke.glade:87
msgid "Region:"
msgstr "Region:"

#: pyanaconda/ui/gui/spokes/datetime_spoke.glade:120
msgid "City:"
msgstr "Miasto:"

#: pyanaconda/ui/gui/spokes/datetime_spoke.glade:164
msgid "Network Time"
msgstr "Czas sieciowy"

#: pyanaconda/ui/gui/spokes/datetime_spoke.glade:257
#: pyanaconda/ui/gui/spokes/datetime_spoke.glade:289
msgid "00"
msgstr "00"

#: pyanaconda/ui/gui/spokes/datetime_spoke.glade:273
msgid ":"
msgstr ":"

#: pyanaconda/ui/gui/spokes/datetime_spoke.glade:401
msgid " "
msgstr " "

#: pyanaconda/ui/gui/spokes/datetime_spoke.glade:417
msgid "PM"
msgstr "PM"

#: pyanaconda/ui/gui/spokes/datetime_spoke.glade:461
msgid "24-hour"
msgstr "24-godzinny"

#: pyanaconda/ui/gui/spokes/datetime_spoke.glade:479
msgid "AM/PM"
msgstr "12-godzinny"

#: pyanaconda/ui/gui/spokes/datetime_spoke.glade:646
msgid "Add and mark for usage NTP servers"
msgstr "Proszę dodać i oznaczyć serwery NTP do użycia"

#: pyanaconda/ui/gui/spokes/datetime_spoke.glade:727
msgid "Hostname"
msgstr "Nazwa komputera"

#: pyanaconda/ui/gui/spokes/datetime_spoke.glade:742
msgid "Working"
msgstr "Działanie"

#: pyanaconda/ui/gui/spokes/datetime_spoke.glade:750
msgid "Use"
msgstr "Użyj"

#: pyanaconda/ui/gui/spokes/network.glade:78
msgid ""
"We'll need network access to fetch information about your location and to "
"make software\n"
"updates available for you."
msgstr ""
"Wymagany jest dostęp sieciowy, aby uzyskać informacje o położeniu i\n"
"udostępnić aktualizacje oprogramowania."

#: pyanaconda/ui/gui/spokes/network.glade:321
#: pyanaconda/ui/gui/spokes/network.glade:674
msgid "Hardware Address"
msgstr "Adres sprzętowy"

#: pyanaconda/ui/gui/spokes/network.glade:335
#: pyanaconda/ui/gui/spokes/network.glade:688
#: pyanaconda/ui/gui/spokes/network.glade:1176
msgid "Speed"
msgstr "Prędkość"

#: pyanaconda/ui/gui/spokes/network.glade:377
msgid "Subnet Mask"
msgstr "Maska podsieci"

#: pyanaconda/ui/gui/spokes/network.glade:391
#: pyanaconda/ui/gui/spokes/network.glade:730
#: pyanaconda/ui/gui/spokes/network.glade:1349
msgid "Default Route"
msgstr "Domyślna trasa"

#: pyanaconda/ui/gui/spokes/network.glade:406
#: pyanaconda/ui/gui/spokes/network.glade:745
#: pyanaconda/ui/gui/spokes/network.glade:1364
msgid "DNS"
msgstr "DNS"

#: pyanaconda/ui/gui/spokes/network.glade:553
msgid "C_onfigure..."
msgstr "Sk_onfiguruj..."

#: pyanaconda/ui/gui/spokes/network.glade:867
msgid "Security"
msgstr "Zabezpieczenia"

#: pyanaconda/ui/gui/spokes/network.glade:881
msgid "_Network Name"
msgstr "_Nazwa sieci"

#: pyanaconda/ui/gui/spokes/network.glade:897
msgid "Network Name"
msgstr "Nazwa sieci"

#: pyanaconda/ui/gui/spokes/network.glade:911
msgid "Security Key"
msgstr "Klucz zabezpieczeń"

#: pyanaconda/ui/gui/spokes/network.glade:1001
msgid "_Use as Hotspot..."
msgstr "_Użyj jako hotspot..."

#: pyanaconda/ui/gui/spokes/network.glade:1020
msgid "_Stop Hotspot..."
msgstr "_Zatrzymaj hotspot"

#: pyanaconda/ui/gui/spokes/network.glade:1039
#: pyanaconda/ui/gui/spokes/network.glade:1391
#: pyanaconda/ui/gui/spokes/network.glade:1676
msgid "_Configure..."
msgstr "S_konfiguruj..."

#: pyanaconda/ui/gui/spokes/network.glade:1162
msgid "IMEI"
msgstr "IMEI"

#: pyanaconda/ui/gui/spokes/network.glade:1190
msgid "Provider"
msgstr "Dostawca"

#: pyanaconda/ui/gui/spokes/network.glade:1504
msgid "VPN Type"
msgstr "Typ VPN"

#: pyanaconda/ui/gui/spokes/network.glade:1518
msgid "Gateway"
msgstr "Brama"

#: pyanaconda/ui/gui/spokes/network.glade:1532
msgid "Group Name"
msgstr "Nazwa grupy"

#: pyanaconda/ui/gui/spokes/network.glade:1546
msgid "Group Password"
msgstr "Hasło grupy"

#: pyanaconda/ui/gui/spokes/network.glade:1560
msgid "Username"
msgstr "Nazwa użytkownika"

#: pyanaconda/ui/gui/spokes/network.glade:1803
msgid "_Method"
msgstr "_Metoda"

#: pyanaconda/ui/gui/spokes/network.glade:1831
msgid "_Configuration URL"
msgstr "Adres URL _konfiguracji"

#: pyanaconda/ui/gui/spokes/network.glade:1861
msgid "_HTTP Proxy"
msgstr "Pośrednik _HTTP"

#: pyanaconda/ui/gui/spokes/network.glade:1877
msgid "H_TTPS Proxy"
msgstr "Pośrednik H_TTPS"

#: pyanaconda/ui/gui/spokes/network.glade:1893
msgid "_FTP Proxy"
msgstr "Pośrednik _FTP"

#: pyanaconda/ui/gui/spokes/network.glade:1909
msgid "_Socks Host"
msgstr "Komputer _SOCKS"

#: pyanaconda/ui/gui/spokes/network.glade:2153
msgid "Unlock"
msgstr "Odblokuj"

#: pyanaconda/ui/gui/spokes/network.glade:2173
msgid "Air_plane Mode"
msgstr "Tryb _samolotowy"

#: pyanaconda/ui/gui/spokes/network.glade:2207
msgid "Hostname:"
msgstr "Nazwa komputera:"

#: pyanaconda/ui/gui/spokes/software.glade:86
msgid "Choose your environment"
msgstr "Wybór środowiska"

#: pyanaconda/ui/gui/spokes/software.glade:106
msgid "Choose your add-ons"
msgstr "Wybór dodatków"

#: pyanaconda/ui/gui/spokes/software.glade:143
#: pyanaconda/ui/gui/spokes/software.glade:158
#: pyanaconda/ui/gui/spokes/software.glade:215
#: pyanaconda/ui/gui/spokes/software.glade:230
msgid "column"
msgstr "kolumna"

#: pyanaconda/ui/gui/spokes/software.glade:266
msgid "_Add custom add-on..."
msgstr "_Dodaj własny dodatek..."

#: pyanaconda/ui/gui/spokes/storage.glade:26
#: pyanaconda/ui/gui/spokes/storage.glade:221
msgid "Cancel & _add more disks"
msgstr "_Anuluj i dodaj więcej dysków"

#: pyanaconda/ui/gui/spokes/storage.glade:79
#: pyanaconda/ui/gui/spokes/storage.glade:293
#: pyanaconda/ui/gui/spokes/storage.glade:650
msgid "INSTALLATION OPTIONS"
msgstr "OPCJE INSTALACJI"

#: pyanaconda/ui/gui/spokes/storage.glade:96
#: pyanaconda/ui/gui/spokes/storage.glade:327
#: pyanaconda/ui/gui/spokes/storage.glade:806
msgid "Here we'll describe what your options are."
msgstr "Tutaj zostaną opisane dostępne opcje."

#: pyanaconda/ui/gui/spokes/storage.glade:122
#: pyanaconda/ui/gui/spokes/storage.glade:477
msgid "<b>Partition type:</b>"
msgstr "<b>Typ partycji:</b>"

#: pyanaconda/ui/gui/spokes/storage.glade:155
#: pyanaconda/ui/gui/spokes/storage.glade:510
msgid "Partition scheme configuration"
msgstr "Konfiguracja schematu partycji"

#: pyanaconda/ui/gui/spokes/storage.glade:170
msgid "Let me customize the _partitioning of the disks instead."
msgstr "_Ręczne dostosowanie partycjonowania dysków."

#: pyanaconda/ui/gui/spokes/storage.glade:238
#: pyanaconda/ui/gui/spokes/storage.glade:594
msgid "_Modify software selection"
msgstr "_Modyfikuj wybór oprogramowania"

#: pyanaconda/ui/gui/spokes/storage.glade:242
#: pyanaconda/ui/gui/spokes/storage.glade:598
msgid "Please wait... software metadata still loading."
msgstr "Proszę czekać, metadane oprogramowania są jeszcze wczytywane."

#: pyanaconda/ui/gui/spokes/storage.glade:257
msgid "Reclaim _space"
msgstr "_Odzyskaj miejsce"

#: pyanaconda/ui/gui/spokes/storage.glade:310
#: pyanaconda/ui/gui/spokes/storage.glade:666
msgid ""
"Here we'll describe how much space is needed for the current software "
"selection."
msgstr ""
"Tutaj zostanie opisane, ile miejsca jest wymagane dla obecnego wyboru "
"oprogramowania."

#: pyanaconda/ui/gui/spokes/storage.glade:353
#: pyanaconda/ui/gui/spokes/storage.glade:692
msgid "disk free"
msgstr "wolny dysk"

#: pyanaconda/ui/gui/spokes/storage.glade:371
#: pyanaconda/ui/gui/spokes/storage.glade:709
msgid "Free space available for use."
msgstr "Wolne miejsce możliwe do użycia."

#: pyanaconda/ui/gui/spokes/storage.glade:389
#: pyanaconda/ui/gui/spokes/storage.glade:727
msgid "fs free"
msgstr "wolny system plików"

#: pyanaconda/ui/gui/spokes/storage.glade:407
#: pyanaconda/ui/gui/spokes/storage.glade:745
msgid "Free space unavailable but reclaimable from existing partitions."
msgstr ""
"Niedostępne wolne miejsce, ale możliwe do odzyskania z istniejących partycji."

#: pyanaconda/ui/gui/spokes/storage.glade:425
#: pyanaconda/ui/gui/spokes/storage.glade:763
msgid "disks size"
msgstr "wielkość dysku"

#: pyanaconda/ui/gui/spokes/storage.glade:443
#: pyanaconda/ui/gui/spokes/storage.glade:781
msgid "Space in selected disks reclaimable by deleting existing partitions."
msgstr ""
"Miejsce na wybranych dyskach możliwe do odzyskania przez usunięcie "
"istniejących partycji."

#: pyanaconda/ui/gui/spokes/storage.glade:525
msgid "I don't need help; let me customize disk _partitioning."
msgstr "Ręczne _dostosowanie partycjonowania dysków."

#: pyanaconda/ui/gui/spokes/storage.glade:577
msgid "_Quit installer"
msgstr "Za_kończ instalator"

#: pyanaconda/ui/gui/spokes/storage.glade:891
msgid "Local Standard Disks"
msgstr "Standardowe, lokalne dyski"

#: pyanaconda/ui/gui/spokes/storage.glade:972
msgid "Full disk summary and options..."
msgstr "Pełne podsumowanie dysków i opcje..."

#: pyanaconda/ui/gui/spokes/storage.glade:997
#: pyanaconda/ui/gui/spokes/custom.glade:1900
msgid "summary"
msgstr "podsumowanie"

#: pyanaconda/ui/gui/spokes/storage.glade:1011
msgid "Encrypt my data. I'll set a passphrase later."
msgstr "Zaszyfrowanie danych. Hasło zostanie ustawione później."

#: pyanaconda/ui/gui/spokes/keyboard.glade:68
msgid "ADD A KEYBOARD LAYOUT"
msgstr "DODAJ UKŁAD KLAWIATURY"

#: pyanaconda/ui/gui/spokes/keyboard.glade:85
msgid "You may add a keyboard layout by selecting it below:"
msgstr "Można dodać układ klawiatury przez wybranie go poniżej:"

#: pyanaconda/ui/gui/spokes/keyboard.glade:121
msgid "Name"
msgstr "Nazwa"

#: pyanaconda/ui/gui/spokes/keyboard.glade:182
msgid "KEYBOARD LAYOUT"
msgstr "UKŁAD KLAWIATURY"

#: pyanaconda/ui/gui/spokes/keyboard.glade:223
msgid ""
"Which keyboard layouts would you like to use on this system?  You may move "
"any layout to the top of the list to select it as the default."
msgstr ""
"Których układów klawiatury używać w tym systemie? Można przesunąć dowolny "
"układ na górę listy, aby wybrać go jako domyślny."

#: pyanaconda/ui/gui/spokes/keyboard.glade:262
msgid ""
"Changes here will only apply to the installed system. Use the desktop's tool "
"to configure the keyboard for the installation process."
msgstr ""
"Zmiany tutaj wprowadzone będą miały zastosowanie tylko do zainstalowanego "
"systemu. Proszę użyć narzędzia środowiska, aby skonfigurować klawiaturę dla "
"procesu instalacji."

#: pyanaconda/ui/gui/spokes/keyboard.glade:319
msgid "name"
msgstr "nazwa"

#: pyanaconda/ui/gui/spokes/keyboard.glade:355
msgid "Add layout"
msgstr "Dodaj układ"

#: pyanaconda/ui/gui/spokes/keyboard.glade:370
msgid "Remove layout"
msgstr "Usuń układ"

#: pyanaconda/ui/gui/spokes/keyboard.glade:385
msgid "Move selected layout up"
msgstr "Przenieś zaznaczony układ w górę"

#: pyanaconda/ui/gui/spokes/keyboard.glade:400
msgid "Move selected layout down"
msgstr "Przenieś zaznaczony układ w dół"

#: pyanaconda/ui/gui/spokes/keyboard.glade:416
msgid "Preview layout"
msgstr "Podgląd układu"

#: pyanaconda/ui/gui/spokes/keyboard.glade:453
msgid "Test the selected layout below:"
msgstr "Poniżej można przetestować wybrany układ:"

#: pyanaconda/ui/gui/spokes/keyboard.glade:492
msgid "Alt + Shift to switch layouts."
msgstr "Naciśnięcie klawiszy Alt + Shift przełącza układy."

#: pyanaconda/ui/gui/spokes/keyboard.glade:506
msgid "_Options"
msgstr "_Opcje"

#: pyanaconda/ui/gui/spokes/keyboard.glade:623
msgid "LAYOUT SWITCHING OPTIONS"
msgstr "OPCJE PRZEŁĄCZANIA UKŁADU"

#: pyanaconda/ui/gui/spokes/keyboard.glade:640
msgid ""
"Which combination(s) would you prefer for switching between keyboard layouts?"
msgstr ""
"Których klawiszy skrótów używać do przełączania między układami klawiatury?"

#: pyanaconda/ui/gui/spokes/keyboard.glade:676
msgid "use"
msgstr "użycie"

#: pyanaconda/ui/gui/spokes/keyboard.glade:689
msgid "description"
msgstr "opis"

#: pyanaconda/ui/gui/spokes/password.glade:52
msgid "Root Password:"
msgstr "Hasło roota:"

#: pyanaconda/ui/gui/spokes/password.glade:67
#: pyanaconda/ui/gui/spokes/lib/passphrase.glade:127
msgid "Confirm:"
msgstr "Potwierdzenie:"

#: pyanaconda/ui/gui/spokes/password.glade:123
msgid ""
"The root account is used for administering the system.  Enter a password for "
"the root user."
msgstr ""
"Konto root jest wykorzystywane do administrowania systemem. Proszę podać "
"hasło dla użytkownika root."

#: pyanaconda/ui/gui/spokes/source.glade:35
#: pyanaconda/ui/gui/spokes/source.glade:607
msgid "_Add"
msgstr "_Dodaj"

#: pyanaconda/ui/gui/spokes/source.glade:66
msgid "ADD A CUSTOM ADD-ON"
msgstr "DODAJ WŁASNY DODATEK"

#: pyanaconda/ui/gui/spokes/source.glade:82
msgid "Where is the yum repository for your add-on?"
msgstr "Gdzie znajduje się repozytorium yum dla dodatku?"

#: pyanaconda/ui/gui/spokes/source.glade:140
msgid "Repository"
msgstr "Repozytorium"

#: pyanaconda/ui/gui/spokes/source.glade:189
msgid "Fedora People Repositories"
msgstr "Repozytoria Fedora People"

#: pyanaconda/ui/gui/spokes/source.glade:306
msgid "Standard YUM repository:"
msgstr "Standardowe repozytorium programu yum:"

#: pyanaconda/ui/gui/spokes/source.glade:439 widgets/src/SpokeWindow.c:46
msgid "_Done"
msgstr "_Gotowe"

#: pyanaconda/ui/gui/spokes/source.glade:472
msgid "MEDIA VERIFICATION"
msgstr "SPRAWDZANIE NOŚNIKA"

#: pyanaconda/ui/gui/spokes/source.glade:499
msgid "Verifying media, please wait..."
msgstr "Sprawdzanie nośnika, proszę czekać..."

#: pyanaconda/ui/gui/spokes/source.glade:635
msgid "_Enable HTTP Proxy"
msgstr "_Włączenie pośrednika HTTP"

#: pyanaconda/ui/gui/spokes/source.glade:668
msgid "<b>Proxy URL</b>"
msgstr "<b>Adres URL pośrednika</b>"

#: pyanaconda/ui/gui/spokes/source.glade:683
msgid "<span size=\"small\"><b>Example:</b> squid.mysite.org:3128</span>"
msgstr ""
"<span size=\"small\"><b>Przykład:</b> squid.mojawitryna.org:3128</span>"

#: pyanaconda/ui/gui/spokes/source.glade:716
msgid "_Use Authentication"
msgstr "_Użycie uwierzytelniania"

#: pyanaconda/ui/gui/spokes/source.glade:743
msgid "<b>Username</b>"
msgstr "<b>Nazwa użytkownika</b>"

#: pyanaconda/ui/gui/spokes/source.glade:758
msgid "<b>Password</b>"
msgstr "<b>Hasło</b>"

#: pyanaconda/ui/gui/spokes/source.glade:869
msgid "Which installation source would you like to use?"
msgstr "Którego źródła instalacji użyć?"

#: pyanaconda/ui/gui/spokes/source.glade:883
msgid "_Auto-detected install media:"
msgstr "_Automatyczne wykrycie nośnika instalacji:"

#: pyanaconda/ui/gui/spokes/source.glade:938
#: pyanaconda/ui/gui/spokes/source.glade:1031
msgid "_Verify"
msgstr "_Sprawdź"

#: pyanaconda/ui/gui/spokes/source.glade:961
msgid "_ISO file:"
msgstr "Plik _ISO:"

#: pyanaconda/ui/gui/spokes/source.glade:988
msgid "Device:"
msgstr "Urządzenie:"

#: pyanaconda/ui/gui/spokes/source.glade:1016
msgid "_Choose an ISO"
msgstr "_Wybierz plik ISO"

#: pyanaconda/ui/gui/spokes/source.glade:1054
msgid "On the _network:"
msgstr "W _sieci:"

#: pyanaconda/ui/gui/spokes/source.glade:1129
msgid "_Proxy setup..."
msgstr "_Ustawienia pośrednika..."

#: pyanaconda/ui/gui/spokes/source.glade:1147
msgid "This URL refers to a _mirror list."
msgstr "Ten adres URL odnosi się do listy _serwerów lustrzanych."

#: pyanaconda/ui/gui/spokes/source.glade:1182
msgid "NFS mount options:"
msgstr "Opcje montowania NFS:"

#: pyanaconda/ui/gui/spokes/source.glade:1195
#: pyanaconda/ui/gui/spokes/source.glade:1196
msgid "This field is optional."
msgstr "To pole jest opcjonalne."

#: pyanaconda/ui/gui/spokes/welcome.glade:21
msgid "_Get me out of here!"
msgstr "_Zabierz mnie stąd!"

#: pyanaconda/ui/gui/spokes/welcome.glade:35
msgid "_I accept my fate."
msgstr "_Przyjmuję swój los."

#: pyanaconda/ui/gui/spokes/welcome.glade:66
msgid "This is unstable, pre-release software."
msgstr "To jest niestabilne, testowe oprogramowanie."

#: pyanaconda/ui/gui/spokes/welcome.glade:99
msgid ""
"You wake up inside an OS installer in Timbuktu, and it's six months in the "
"future.  But, there are bugs.  Bugs everywhere.  Bugs you must live with.  "
"This OS of the future isn't a stable OS you can rely on.  It's for testing "
"purposes only."
msgstr ""
"Budzisz się wewnątrz instalatora systemu operacyjnego w Timbuktu, sześć "
"miesięcy później. Widzisz błędy. Wszędzie same błędy. Błędy, z którymi "
"musisz żyć. Ten system przyszłości nie jest stabilnym systemem, na którym "
"można polegać. Jest przeznaczony tylko do testowania."

#: pyanaconda/ui/gui/spokes/welcome.glade:251
msgid "WELCOME TO %s %s."
msgstr "WITAJ W SYSTEMIE %s %s."

#: pyanaconda/ui/gui/spokes/welcome.glade:269
msgid "What language would you like to use during the installation process?"
msgstr "Jakiego języka używać podczas instalacji?"

#: pyanaconda/ui/gui/spokes/welcome.glade:319
msgid "nativeName"
msgstr "polski"

#: pyanaconda/ui/gui/spokes/welcome.glade:335
msgid "englishName"
msgstr "Polish"

#: pyanaconda/ui/gui/spokes/welcome.glade:389
msgid "Set _keyboard to default layout for selected language."
msgstr "Ustawienie _klawiatury na domyślny układ dla wybranego języka."

#: pyanaconda/ui/gui/spokes/custom.glade:8
#: pyanaconda/ui/gui/spokes/custom.glade:1741
msgid "Help"
msgstr "Pomoc"

#: pyanaconda/ui/gui/spokes/custom.glade:106
msgid "_Add mount point"
msgstr "_Dodaj punkt montowania"

#: pyanaconda/ui/gui/spokes/custom.glade:139
msgid "<b>Desired Capacity:</b>"
msgstr "<b>Żądana pojemność:</b>"

#: pyanaconda/ui/gui/spokes/custom.glade:154
#: pyanaconda/ui/gui/spokes/custom.glade:558
msgid "<b>Mount Point:</b>"
msgstr "<b>Punkt montowania:</b>"

#: pyanaconda/ui/gui/spokes/custom.glade:170
msgid "ADD A NEW MOUNT POINT"
msgstr "DODAJ NOWY PUNKT MONTOWANIA"

#: pyanaconda/ui/gui/spokes/custom.glade:188
msgid ""
"More customization options are available\n"
"after creating the mount point below."
msgstr ""
"Więcej opcji dostosowania jest dostępnych\n"
"po utworzeniu punktu montowania poniżej."

#: pyanaconda/ui/gui/spokes/custom.glade:206
msgid "eg: \"/\", \"/home\", \"swap\" (minus the quotation marks)"
msgstr "np.: \"/\", \"/home\", \"swap\" (bez cudzysłowów)"

#: pyanaconda/ui/gui/spokes/custom.glade:239
msgid ""
"That mount point is already in\n"
"use.  Try something else?"
msgstr "Ten punkt montowania jest już używany.Proszę spróbować innego."

#: pyanaconda/ui/gui/spokes/custom.glade:265
msgid "eg: \"20 GB\", \"500mb\" (minus the quotation marks)"
msgstr "pn.: \"20 GB\", \"500mb\" (bez cudzysłowów)"

#: pyanaconda/ui/gui/spokes/custom.glade:308
msgid "Are you sure you want to delete all of the data on %s?"
msgstr "Na pewno usunąć wszystkie dane na urządzeniu %s?"

#: pyanaconda/ui/gui/spokes/custom.glade:322
msgid "Delete all other filesystems in the %s root as well."
msgstr "Usunięcie także wszystkich innych systemów plików w roocie %s."

#: pyanaconda/ui/gui/spokes/custom.glade:357
msgid "_Delete It"
msgstr "_Usuń"

#: pyanaconda/ui/gui/spokes/custom.glade:390
msgid "_Back to destination selection"
msgstr "_Wróć do wyboru miejsca docelowego"

#: pyanaconda/ui/gui/spokes/custom.glade:476
msgid ""
"When you create mount points for your %s %s install,\n"
"you'll be able to view their details here."
msgstr ""
"Po utworzeniu punktów montowania dla instalacji\n"
"systemu %s %s ich szczegóły zostaną tutaj wyświetlone."

#: pyanaconda/ui/gui/spokes/custom.glade:499
#: pyanaconda/ui/gui/spokes/custom.glade:1362
#: pyanaconda/ui/gui/spokes/custom.glade:1504
#: pyanaconda/ui/gui/spokes/custom.glade:1591
msgid "Selected Device"
msgstr "Wybrane urządzenie"

#: pyanaconda/ui/gui/spokes/custom.glade:517
#: pyanaconda/ui/gui/spokes/custom.glade:1380
#: pyanaconda/ui/gui/spokes/custom.glade:1522
#: pyanaconda/ui/gui/spokes/custom.glade:1609
msgid "Device description"
msgstr "Opis urządzenia"

#: pyanaconda/ui/gui/spokes/custom.glade:588
msgid "<b>Label:</b>"
msgstr "<b>Etykieta:</b>"

#: pyanaconda/ui/gui/spokes/custom.glade:604
msgid "<b>Desired Capacity (MB):</b>"
msgstr "<b>Żądana pojemność (MB):</b>"

#: pyanaconda/ui/gui/spokes/custom.glade:651
#: pyanaconda/ui/gui/spokes/custom.glade:2036
msgid "Name:"
msgstr "Nazwa:"

#: pyanaconda/ui/gui/spokes/custom.glade:712
msgid "<b>Device Type:</b>"
msgstr "<b>Typ urządzenia:</b>"

#: pyanaconda/ui/gui/spokes/custom.glade:727
msgid "<b>File System:</b>"
msgstr "<b>System plików:</b>"

#: pyanaconda/ui/gui/spokes/custom.glade:773
msgid "_Encrypt"
msgstr "Za_szyfrowane"

#: pyanaconda/ui/gui/spokes/custom.glade:806
#: pyanaconda/ui/gui/spokes/custom.glade:903
#: pyanaconda/ui/gui/spokes/custom.glade:1144
msgid "Redundancy (mirror)"
msgstr "Kopia (mirror)"

#: pyanaconda/ui/gui/spokes/custom.glade:825
#: pyanaconda/ui/gui/spokes/custom.glade:868
#: pyanaconda/ui/gui/spokes/custom.glade:922
#: pyanaconda/ui/gui/spokes/custom.glade:965
#: pyanaconda/ui/gui/spokes/custom.glade:1008
#: pyanaconda/ui/gui/spokes/custom.glade:1052
#: pyanaconda/ui/gui/spokes/custom.glade:1084
#: pyanaconda/ui/gui/spokes/custom.glade:1163
#: pyanaconda/ui/gui/spokes/custom.glade:1206
msgid "(-0Gb)"
msgstr "(-0Gb)"

#: pyanaconda/ui/gui/spokes/custom.glade:850
#: pyanaconda/ui/gui/spokes/custom.glade:947
#: pyanaconda/ui/gui/spokes/custom.glade:1188
msgid "Optimized performance (stripe)"
msgstr "Optymalizacja dla wydajności (stripe)"

#: pyanaconda/ui/gui/spokes/custom.glade:990
msgid "Error detection (parity)"
msgstr "Wykrywanie błędów (parity)"

#: pyanaconda/ui/gui/spokes/custom.glade:1034
msgid "Distributed"
msgstr "Rozproszone"

#: pyanaconda/ui/gui/spokes/custom.glade:1065
msgid "Redundant"
msgstr "Kopia"

#: pyanaconda/ui/gui/spokes/custom.glade:1243
msgid "Reformat"
msgstr "Sformatowanie"

#: pyanaconda/ui/gui/spokes/custom.glade:1262
msgid "Volume Group:"
msgstr "Grupa woluminów:"

#: pyanaconda/ui/gui/spokes/custom.glade:1291
msgid "Modify ..."
msgstr "Modyfikuj..."

#: pyanaconda/ui/gui/spokes/custom.glade:1310
msgid "_Customize..."
msgstr "_Dostosuj..."

#: pyanaconda/ui/gui/spokes/custom.glade:1323
msgid "_Apply Changes"
msgstr "_Zastosuj zmiany"

#: pyanaconda/ui/gui/spokes/custom.glade:1414
msgid ""
"This device is encrypted and cannot be read without a valid passphrase. You "
"may unlock it below."
msgstr ""
"To urządzenie jest zaszyfrowane i nie może zostać odczytane bez prawidłowego "
"hasła. Można odblokować je poniżej."

#: pyanaconda/ui/gui/spokes/custom.glade:1435
#: pyanaconda/ui/gui/spokes/lib/passphrase.glade:97
msgid "Passphrase:"
msgstr "Hasło:"

#: pyanaconda/ui/gui/spokes/custom.glade:1461
msgid "_Unlock"
msgstr "Od_blokuj"

#: pyanaconda/ui/gui/spokes/custom.glade:1556
msgid ""
"This device cannot be edited directly. You can remove it or select a "
"different device."
msgstr ""
"Tego urządzenia nie można modyfikować bezpośrednio. Można je usunąć lub "
"wybrać inne."

#: pyanaconda/ui/gui/spokes/custom.glade:1644
msgid ""
"This %s device is missing %d of %d %s. You can remove it or select a "
"different device."
msgstr "W urządzeniu %s brakuje %d z %d %s. Można je usunąć lub wybrać inne."

#: pyanaconda/ui/gui/spokes/custom.glade:1690
msgid "Add a new mountpoint."
msgstr "Dodaj nowy punkt montowania."

#: pyanaconda/ui/gui/spokes/custom.glade:1691
msgid "addButton"
msgstr "Dodaj"

#: pyanaconda/ui/gui/spokes/custom.glade:1708
msgid "Remove the selected mountpoint(s.)"
msgstr "Usuń zaznaczone punkty montowania."

#: pyanaconda/ui/gui/spokes/custom.glade:1709
msgid "removeButton"
msgstr "Usuń"

#: pyanaconda/ui/gui/spokes/custom.glade:1726
msgid "Configure selected mountpoint."
msgstr "Konfiguruj zaznaczony punkt montowania."

#: pyanaconda/ui/gui/spokes/custom.glade:1727
msgid "configureButton"
msgstr "Skonfiguruj"

#: pyanaconda/ui/gui/spokes/custom.glade:1742
msgid "Click for help"
msgstr "Kliknięcie wyświetli pomoc"

#: pyanaconda/ui/gui/spokes/custom.glade:1790
msgid "<span size=\"small\" foreground=\"white\">AVAILABLE SPACE</span>"
msgstr "<span size=\"small\" foreground=\"white\">DOSTĘPNE MIEJSCE</span>"

#: pyanaconda/ui/gui/spokes/custom.glade:1842
msgid "<span size=\"small\" foreground=\"white\">TOTAL SPACE</span>"
msgstr "<span size=\"small\" foreground=\"white\">RAZEM MIEJSCE</span>"

#: pyanaconda/ui/gui/spokes/custom.glade:1881
msgid "_Finish Partitioning"
msgstr "_Zakończ partycjonowanie"

#: pyanaconda/ui/gui/spokes/custom.glade:1971
msgid "_Save"
msgstr "Zapi_sz"

#: pyanaconda/ui/gui/spokes/custom.glade:2003
msgid "CONFIGURE VOLUME GROUP"
msgstr "SKONFIGURUJ GRUPĘ WOLUMINÓW"

#: pyanaconda/ui/gui/spokes/custom.glade:2018
msgid ""
"Please create a name for your volume group and select at least one disk "
"below."
msgstr ""
"Proszę utworzyć nazwę dla grupy woluminów i wybrać co najmniej jeden dysk "
"poniżej."

#: pyanaconda/ui/gui/spokes/custom.glade:2101
#: pyanaconda/ui/gui/spokes/custom.glade:2302
#: pyanaconda/ui/gui/spokes/lib/cart.glade:138
msgid "Capacity"
msgstr "Pojemność"

#: pyanaconda/ui/gui/spokes/custom.glade:2114
#: pyanaconda/ui/gui/spokes/custom.glade:2315
#: pyanaconda/ui/gui/spokes/lib/cart.glade:151
msgid "Free"
msgstr "Wolne"

#: pyanaconda/ui/gui/spokes/custom.glade:2127
#: pyanaconda/ui/gui/spokes/custom.glade:2328
msgid "ID"
msgstr "Identyfikator"

#: pyanaconda/ui/gui/spokes/custom.glade:2209
msgid "_Select"
msgstr "_Wybierz"

#: pyanaconda/ui/gui/spokes/custom.glade:2241
msgid "CONFIGURE MOUNT POINT"
msgstr "SKONFIGURUJ PUNKT MONTOWANIA"

#: pyanaconda/ui/gui/spokes/custom.glade:2256
msgid "Select one or more disks this device may reside on."
msgstr ""
"Proszę wybrać jeden lub więcej dysków, na których to urządzenie może się "
"znajdować."

#: pyanaconda/ui/gui/spokes/lib/cart.glade:25
#: pyanaconda/ui/gui/spokes/lib/cart.glade:79
msgid "SELECTED DISKS"
msgstr "WYBRANE DYSKI"

#: pyanaconda/ui/gui/spokes/lib/cart.glade:43
msgid "_Close"
msgstr "_Zamknij"

#: pyanaconda/ui/gui/spokes/lib/cart.glade:111
msgid "Boot"
msgstr "Startowe"

#: pyanaconda/ui/gui/spokes/lib/cart.glade:125
msgid "Description"
msgstr "Opis"

#: pyanaconda/ui/gui/spokes/lib/cart.glade:163
msgid "Id"
msgstr "Identyfikator"

#: pyanaconda/ui/gui/spokes/lib/cart.glade:207
msgid "_Remove"
msgstr "_Usuń"

#: pyanaconda/ui/gui/spokes/lib/cart.glade:235
msgid "Disk summary goes here"
msgstr "Tutaj zostanie wyświetlone podsumowanie dysku"

#: pyanaconda/ui/gui/spokes/lib/detailederror.glade:24
msgid "An unknown error occurred during installation.  Details are below."
msgstr ""
"Wystąpił nieznany błąd podczas instalacji. Poniżej znajduje się więcej "
"informacji."

#: pyanaconda/ui/gui/spokes/lib/passphrase.glade:20
msgid "DISK ENCRYPTION PASSPHRASE"
msgstr "HASŁO SZYFROWANIA DYSKU"

#: pyanaconda/ui/gui/spokes/lib/passphrase.glade:52
msgid "Save Passphrase"
msgstr "Zapisz hasło"

#: pyanaconda/ui/gui/spokes/lib/passphrase.glade:77
msgid ""
"You have chosen to encrypt some of your data. You will need to create a "
"passphrase that you will use to access your data when you start your "
"computer."
msgstr ""
"Wybrano zaszyfrowanie części danych. Należy utworzyć hasło, które będzie "
"używane do uzyskiwania dostępu do danych podczas uruchamiania komputera."

#: pyanaconda/ui/gui/spokes/lib/passphrase.glade:212
msgid "keymap warning label"
msgstr "etykieta ostrzeżenia o mapie klawiszy"

#: pyanaconda/ui/gui/spokes/lib/resize.glade:58
msgid "_Reclaim space"
msgstr "_Odzyskaj miejsce"

#: pyanaconda/ui/gui/spokes/lib/resize.glade:92
msgid "RECLAIM DISK SPACE"
msgstr "ODZYSKIWANIE MIEJSCA NA DYSKU"

#: pyanaconda/ui/gui/spokes/lib/resize.glade:109
msgid "Description goes here."
msgstr "Miejsce na opis."

#: pyanaconda/ui/gui/spokes/lib/resize.glade:151
msgid "Filesystem"
msgstr "System plików"

#: pyanaconda/ui/gui/spokes/lib/resize.glade:162
msgid "Reclaimable Space"
msgstr "Miejsce możliwe do odzyskania"

#: pyanaconda/ui/gui/spokes/lib/resize.glade:184
msgid "Action"
msgstr "Działanie"

#: pyanaconda/ui/gui/spokes/lib/resize.glade:249
msgid "<b>%s disks; %s reclaimable space</b> (in filesystems)"
msgstr ""
"<b>%s dyski; %s miejsca możliwego do odzyskania</b> (w systemach plików)"

#: pyanaconda/ui/gui/spokes/lib/resize.glade:264
msgid "Total selected space to reclaim:  <b>%s</b>"
msgstr "Razem wybranego miejsca do odzyskania: <b>%s</b>"

#: pyanaconda/ui/gui/spokes/lib/resize.glade:279
msgid "Installation requires a total of <b>%s</b> for system data."
msgstr "Instalacja wymaga razem <b>%s</b> na dane systemu."

#: pyanaconda/ui/gui/main.glade:80
msgid ""
"Are you sure you wish to quit the\n"
"installation process?"
msgstr "Na pewno zakończyć instalację?"

#: pyanaconda/ui/gui/hubs/summary.glade:11
msgid "INSTALLATION SUMMARY"
msgstr "PODSUMOWANIE INSTALACJI"

#: pyanaconda/ui/gui/hubs/summary.glade:95
msgid "_Begin Installation"
msgstr "_Rozpocznij instalację"

#: pyanaconda/ui/gui/hubs/progress.glade:9
msgid "CONFIGURATION"
msgstr "KONFIGURACJA"

#: pyanaconda/ui/gui/hubs/progress.glade:128
msgid ""
"%s is now successfully installed on your system, but some configuration "
"still needs to be done. \n"
"Finish it and then click the Finish configuration button please."
msgstr ""
"System %s został pomyślnie zainstalowany na komputerze, ale wymagane jest "
"skonfigurowanie jeszcze kilku rzeczy. \n"
"Proszę je skonfigurować, a następnie nacisnąć przycisk Zakończ konfigurację."

#: pyanaconda/ui/gui/hubs/progress.glade:140
msgid "_Finish configuration"
msgstr "Za_kończ konfigurację"

#: pyanaconda/ui/gui/hubs/progress.glade:181
msgid ""
"%s is now successfully installed on your system and ready\n"
"for you to use!  Go ahead and reboot to start using it!"
msgstr ""
"System %s został pomyślnie zainstalowany na komputerze i jest gotowy do "
"użycia. Należy ponownie uruchomić komputer."

#: pyanaconda/ui/gui/hubs/progress.glade:193
msgid "_Reboot"
msgstr "_Uruchom ponownie"

#: widgets/src/BaseWindow.c:85
msgid "DISTRIBUTION INSTALLATION"
msgstr "INSTALACJA DYSTRYBUCJI"

#: widgets/src/BaseWindow.c:86
msgid "SPOKE NAME"
msgstr "NAZWA EKRANU"

#: widgets/src/BaseWindow.c:87
msgid "PRE-RELEASE / TESTING"
msgstr "WYDANIE TESTOWE"

#: widgets/src/DiskOverview.c:50
msgid "New Device"
msgstr "Nowe urządzenie"

#: widgets/src/DiskOverview.c:52
msgid "0 MB"
msgstr "0 MB"

#: widgets/src/MountpointSelector.c:48
msgid "Root"
msgstr "Root"

#: widgets/src/MountpointSelector.c:49
msgid "0 GB"
msgstr "0 GB"

#: widgets/src/SpokeSelector.c:55
msgid "New Selector"
msgstr "Nowy wybór"

#: data/liveinst/gnome/fedora-welcome:64
msgid "Welcome to Fedora"
msgstr "Witaj w Fedorze"

#: data/liveinst/gnome/fedora-welcome:93
msgid "Try Fedora"
msgstr "Wypróbuj Fedorę"

#: data/liveinst/gnome/fedora-welcome:111
msgid ""
"You are currently running Fedora from live media.\n"
"You can install Fedora now, or choose \"Install to Hard Drive\" in the "
"Activities Overview at any later time."
msgstr ""
"Obecnie Fedora jest uruchomiona z nośnika Live.\n"
"Można zainstalować Fedorę teraz lub wybrać \"Instalacja na dysku twardym\" "
"na ekranie podglądu w dowolnym czasie."

#: data/liveinst/gnome/fedora-welcome:129
msgid ""
"You can choose \"Install to Hard Drive\"\n"
"in the Activities Overview at any later time."
msgstr ""
"Można wybrać \"Instalacja na dysku twardym\"\n"
"na ekranie podglądu w dowolnym czasie."

#: data/liveinst/gnome/fedora-welcome:132
msgid "Close"
msgstr "Zamknij"

#: data/liveinst/liveinst.desktop.in.h:1
msgid "Install to Hard Drive"
msgstr "Instalacja na dysku twardym"

#: data/liveinst/liveinst.desktop.in.h:2
msgid "Install"
msgstr "Instalacja"

#: data/liveinst/liveinst.desktop.in.h:3
msgid "Install the live CD to your hard disk"
msgstr "Instalacja obrazu Live na dysku twardym"

#: liveinst:5
msgid "Starting Install to Hard Drive"
msgstr "Rozpoczynanie instalacji na dysku twardym"
